<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE reference PUBLIC "-//OASIS//DTD DITA Reference//EN"
 "../../dtd/reference.dtd">
<!--
Licensed to the Apache Software Foundation (ASF) under one or more
contributor license agreements.  See the NOTICE file distributed with
this work for additional information regarding copyright ownership.
The ASF licenses this file to You under the Apache License, Version 2.0
(the "License"); you may not use this file except in compliance with
the License.  You may obtain a copy of the License at

http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
-->
<reference id="rrefexcept71493" xml:lang="pt-br">
<title>Referência de SQLState e mensagem de erro</title>
<refbody>
<section>
<p>As tabelas a seguir listam os <i>SQLState</i> para as exceções.
As exceções que começam por <i>X</i> são específicas do
<ph conref="refconrefs.dita#prod/productshortname"></ph>.
Deve ser observado que alguns SQLState específicos do cliente da rede poderão
mudar nas versões futuras.</p>
</section>
<section>
<table>
<title>Código de classe 01: Advertência</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row valign="bottom">
<entry colname="col1">SQLSTATE</entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">01003       </entry>
<entry colname="col2">Null values were eliminated from the argument of a column
function.  </entry>
</row>
<row>
<entry colname="col1">0100E        </entry>
<entry colname="col2">XX Attempt to return too many result sets.     </entry>
</row>
<row>
<entry colname="col1">01500</entry>
<entry colname="col2">The constraint &lt;<i>constraintName</i>> on table &lt;<i>tableName</i>>
has been dropped.     </entry>
</row>
<row>
<entry colname="col1">01501</entry>
<entry colname="col2">The view &lt;<i>viewName</i>> has been dropped.    </entry>
</row>
<row>
<entry colname="col1">01502</entry>
<entry colname="col2">The trigger &lt;<i>triggerName</i>> on table &lt;<i>tableName</i>>
has been dropped. </entry>
</row>
<row>
<entry colname="col1">01503</entry>
<entry colname="col2">The column &lt;<i>columnName</i>> on table &lt;<i>tableName</i>>
has been modified by adding a not null constraint. </entry>
</row>
<row>
<entry colname="col1">01504  </entry>
<entry colname="col2">The new index is a duplicate of an existing index: &lt;<i>index</i>>.
   </entry>
</row>
<row>
<entry colname="col1">01505</entry>
<entry colname="col2">The value &lt;<i>valueName</i>> may be truncated.</entry>
</row>
<row>
<entry colname="col1">01522</entry>
<entry colname="col2">The newly defined synonym '&lt;<i>synonymName</i>>'
resolved to the object '&lt;<i>objectName</i>>' which is currently undefined.</entry>
</row>
<row>
<entry colname="col1">01J01</entry>
<entry colname="col2">Database '&lt;<i>databaseName</i>>' not created, connection
made to existing database instead. </entry>
</row>
<row>
<entry colname="col1">01J02</entry>
<entry colname="col2">Scroll sensitive cursors are not currently implemented. </entry>
</row>
<row>
<entry colname="col1">01J03</entry>
<entry colname="col2">Scroll sensitive and scroll insensitive updatable ResultSets
are not currently implemented. </entry>
</row>
<row>
<entry colname="col1">01J04</entry>
<entry colname="col2">The class '&lt;<i>className</i>>' for column '&lt;<i>columnName</i>>'
does not implement java.io.Serializable or  java.sql.SQLData. Instances must
implement one of these interfaces to allow them to be stored.</entry>
</row>
<row>
<entry colname="col1">01J05</entry>
<entry colname="col2">Database upgrade succeeded. The upgraded database is
now ready for use.  Revalidating stored prepared statements failed. See next
exception for details about failure. </entry>
</row>
<row>
<entry colname="col1">01J06</entry>
<entry colname="col2">ResultSet not updatable. Query does not qualify to generate
an updatable ResultSet.          </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 04: Autenticação no banco de dados</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">04501</entry>
<entry colname="col2">Database connection refused. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 07: Erro de SQL dinâmico</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">07000</entry>
<entry colname="col2">At least one parameter to the current statement is uninitialized.
       </entry>
</row>
<row>
<entry colname="col1">07004</entry>
<entry colname="col2">Parameter &lt;<i>parameterName</i>> is a &lt;<i>procedureName</i>>
procedure parameter and must be registered with CallableStatement.registerOutParameter
before execution. </entry>
</row>
<row>
<entry colname="col1"> 07009</entry>
<entry colname="col2">No input parameters.        </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 08: Exceção de conexão</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">08000  </entry>
<entry colname="col2">Connection closed by unknown interrupt.   </entry>
</row>
<row>
<entry colname="col1">08003</entry>
<entry colname="col2">No current connection. </entry>
</row>
<row>
<entry colname="col1">08004</entry>
<entry colname="col2">Connection refused : &lt;<i>connectionName</i>>
         </entry>
</row>
<row>
<entry colname="col1">08006</entry>
<entry colname="col2">Database '&lt;<i>databaseName</i>>'</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 0A: Funcionalidade não suportada</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">0A000</entry>
<entry colname="col2">Feature not implemented: &lt;<i>featureName</i>>.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 21: Violação de cardinalidade</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">21000</entry>
<entry colname="col2">Scalar subquery is only allowed to return a single row.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 22: Exceção de dado</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">2200L</entry>
<entry colname="col2">XMLPARSE operand is not an XML document; see next exception
for details.</entry>
</row>
<row>
<entry colname="col1">22001</entry>
<entry colname="col2">A truncation error was encountered trying to shrink &lt;<i>value</i>>
'&lt;<i>value</i>>' to length &lt;<i>value</i>></entry>
</row>
<row valign="top">
<entry colname="col1">22003 </entry>
<entry colname="col2">The resulting value is outside the range for the data
type &lt;<i>datatypeName</i>>. </entry>
</row>
<row>
<entry colname="col1">22005</entry>
<entry colname="col2">An attempt was made to get a data value of type '&lt;<i>typeName</i>>'
from a data value of type '&lt;<i>typeName</i>>'. </entry>
</row>
<row>
<entry colname="col1">22007</entry>
<entry colname="col2">The string representation of a datetime value is out
of range.  </entry>
</row>
<row>
<entry colname="col1">22007</entry>
<entry colname="col2">The syntax of the string representation of a datetime
value is incorrect. </entry>
</row>
<row>
<entry colname="col1">22008</entry>
<entry colname="col2">'&lt;<i>argument</i>>' is an invalid argument to the &lt;<i>functionName</i>>
function.</entry>
</row>
<row>
<entry colname="col1">22011</entry>
<entry colname="col2">The second or third argument of the SUBSTR function
is out of range. </entry>
</row>
<row>
<entry colname="col1">22012</entry>
<entry colname="col2">Attempt to divide by zero.   </entry>
</row>
<row>
<entry colname="col1">22013</entry>
<entry colname="col2">Attempt to take the square root of a negative number,
'&lt;<i>number</i>>'.      </entry>
</row>
<row>
<entry colname="col1">22014</entry>
<entry colname="col2">The start position for LOCATE is invalid; it must be
a positive integer. The index  to start the search from is '&lt;<i>index</i>>'.
 The string to search for is '&lt;<i>index</i>>'.  The string to search from
is '&lt;<i>index</i>>'. </entry>
</row>
<row>
<entry colname="col1">22015                       </entry>
<entry colname="col2">The '&lt;<i>functionName</i>>' function is not allowed
on the following set of types.  First operand is of type '&lt;<i>typeName</i>>'.
Second operand is of type '&lt;<i>typeName</i>>'.  Third operand (start position)
is of type '&lt;<i>typeName</i>>'. </entry>
</row>
<row>
<entry colname="col1">22018</entry>
<entry colname="col2">Invalid character string format for type &lt;<i>typeName</i>>. </entry>
</row>
<row>
<entry colname="col1">22019</entry>
<entry colname="col2">Invalid escape sequence, '&lt;<i>sequenceName</i>>'.
The escape string must be exactly one character.  It cannot be a null or more
than one character. </entry>
</row>
<row>
<entry colname="col1">22025</entry>
<entry colname="col2">Escape character must be followed by escape character,
_, or %. It cannot be followed by any other character or be at the end of
the pattern.   </entry>
</row>
<row>
<entry colname="col1">22027</entry>
<entry colname="col2">The built-in TRIM() function only supports a single
trim character.  The LTRIM() and RTRIM() built-in functions support multiple
trim characters.</entry>
</row>
<row>
<entry colname="col1">22501</entry>
<entry colname="col2">An ESCAPE clause of NULL returns undefined results and
is not allowed.              </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 23: Violação de restrição</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">23502   </entry>
<entry colname="col2">Column '&lt;<i>columnName</i>>'  cannot accept a NULL
value. </entry>
</row>
<row>
<entry colname="col1">23503</entry>
<entry colname="col2">&lt;<i>value</i>> on table '&lt;<i>tableName</i>>' caused
a violation of foreign key constraint '&lt;<i>constraintName</i>>' for key &lt;<i>keyName</i>>.
The statement has been rolled back.</entry>
</row>
<row>
<entry colname="col1">23505</entry>
<entry colname="col2">The statement was aborted because it would have caused
a duplicate key value in a unique or primary key constraint or unique index
identified by '&lt;<i>value</i>>' defined on '&lt;<i>value</i>>'.  </entry>
</row>
<row>
<entry colname="col1">23513</entry>
<entry colname="col2">The check constraint '&lt;<i>constraintName</i>>' was
violated while performing an INSERT or UPDATE on table '&lt;<i>tableName</i>>'.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 24: Estado do cursor inválido</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">24000</entry>
<entry colname="col2">Invalid cursor state - no current row. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 25: Estado da transação inválido</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">25000</entry>
<entry colname="col2">Invalid transaction state. </entry>
</row>
<row>
<entry colname="col1">25501</entry>
<entry colname="col2">Unable to set the connection read-only property in an
active transaction.   </entry>
</row>
<row>
<entry colname="col1">25502</entry>
<entry colname="col2">An SQL data change is not permitted for a read-only
connection, user or database. </entry>
</row>
<row>
<entry colname="col1">25503</entry>
<entry colname="col2">DDL is not permitted for a read-only connection, user
or database.  </entry>
</row>
<row>
<entry colname="col1"> 25505</entry>
<entry colname="col2">A read-only user or a user in a read-only database is
not permitted to disable read-only mode on a connection.                  </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 28: Especificação de autorização inválida</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">28501  </entry>
<entry colname="col2">Invalid database authorization property '&lt;<i>value</i>>=&lt;<i>value</i>>'.
 </entry>
</row>
<row>
<entry colname="col1">28502  </entry>
<entry colname="col2">The user name '&lt;<i>userName</i>>' is not valid.</entry>
</row>
<row>
<entry colname="col1">28503</entry>
<entry colname="col2">User(s) '&lt;<i>userName</i>>' must not be in both read-only
and full-access authorization lists. </entry>
</row>
<row>
<entry colname="col1">28504</entry>
<entry colname="col2">Repeated user(s) '&lt;<i>userName</i>>' in access list
'&lt;<i>listName</i>>'. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 38: Exceção de função externa</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">38000</entry>
<entry colname="col2">The exception '&lt;<i>exception</i>>' was thrown while
evaluating an expression. </entry>
</row>
<row>
<entry colname="col1">38001  </entry>
<entry colname="col2">The external routine is not allowed to execute SQL statements.
          </entry>
</row>
<row>
<entry colname="col1">38002</entry>
<entry colname="col2">The routine attempted to modify data, but the routine
was not defined as MODIFIES SQL DATA.</entry>
</row>
<row>
<entry colname="col1">38004</entry>
<entry colname="col2">The routine attempted to read data, but the routine
was not defined as READS SQL DATA.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 39: Exceção de função externa</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">39004</entry>
<entry colname="col2">A NULL value cannot be passed to a method which takes
a  parameter of primitive type '&lt;<i>type</i>>'.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 3B: SAVEPOINT inválido</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">3B001</entry>
<entry colname="col2">SAVEPOINT, &lt;<i>savepoint</i>> does not  exist or
is not active in the current transaction.  </entry>
</row>
<row>
<entry colname="col1">3B002</entry>
<entry colname="col2">The maximum number of savepoints has been reached. </entry>
</row>
<row>
<entry colname="col1"> 3B501</entry>
<entry colname="col2">A SAVEPOINT with the passed name already exists in the
current transaction. </entry>
</row>
<row>
<entry colname="col1">3B502</entry>
<entry colname="col2">A RELEASE or ROLLBACK TO SAVEPOINT was specified, but
the savepoint does not exist.      </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 40: Transação desfeita</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">40001</entry>
<entry colname="col2">A lock could not be obtained due to a deadlock, cycle
of locks and waiters is: &lt;<i>value</i>>. The selected victim is XID : &lt;<i>value</i>>. </entry>
</row>
<row>
<entry colname="col1">40XC0</entry>
<entry colname="col2">Dead statement. This may be caused by catching a transaction
severity error inside this statement. </entry>
</row>
<row>
<entry colname="col1">40XD0</entry>
<entry colname="col2">Container has been closed. </entry>
</row>
<row>
<entry colname="col1">40XD1</entry>
<entry colname="col2">Container was opened in read-only mode.   </entry>
</row>
<row>
<entry colname="col1">40XD2</entry>
<entry colname="col2">Container &lt;<i>containerName</i>> cannot be opened;
it either has been dropped or does not exist.</entry>
</row>
<row>
<entry colname="col1">40XL1</entry>
<entry colname="col2">A lock could not be obtained within the time requested.</entry>
</row>
<row>
<entry colname="col1">40XL2</entry>
<entry colname="col2">A lock could not be obtained within the time requested.
 The lockTable dump is: &lt;<i>tableDump</i>>.</entry>
</row>
<row>
<entry colname="col1">40XT0</entry>
<entry colname="col2">An internal error was identified by RawStore module. </entry>
</row>
<row>
<entry colname="col1">40XT1</entry>
<entry colname="col2">An exception was thrown during transaction commit.</entry>
</row>
<row>
<entry colname="col1">40XT2</entry>
<entry colname="col2">An exception was thrown during rollback of a SAVEPOINT. </entry>
</row>
<row>
<entry colname="col1">40XT4</entry>
<entry colname="col2">An attempt was made to close a transaction that was
still active. The transaction has been aborted. </entry>
</row>
<row>
<entry colname="col1">40XT5</entry>
<entry colname="col2">Exception thrown during an internal transaction. </entry>
</row>
<row>
<entry colname="col1">40XT6</entry>
<entry colname="col2">Database is in quiescent state, cannot activate transaction.
 Please wait for a moment until it exits the inactive state. </entry>
</row>
<row>
<entry colname="col1">40XT7</entry>
<entry colname="col2">Operation is not supported in an internal transaction.
 </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe 42: Erro de sintaxe ou violação de regra de acesso</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">42000</entry>
<entry colname="col2">Syntax error or access rule violation; see additional
errors for details. </entry>
</row>
<row>
<entry colname="col1">42601</entry>
<entry colname="col2">ALTER TABLE statement cannot add an IDENTITY column
to a table.  </entry>
</row>
<row>
<entry colname="col1">42601</entry>
<entry colname="col2">In an ALTER TABLE statement, the column '&lt;<i>columnName</i>>'
has been specified as NOT NULL and either the DEFAULT clause was not specified
or was specified as DEFAULT NULL. </entry>
</row>
<row>
<entry colname="col1">42605</entry>
<entry colname="col2">The number of arguments for function '&lt;<i>functionName</i>>'
is incorrect.  </entry>
</row>
<row>
<entry colname="col1">42606</entry>
<entry colname="col2">An invalid hexadecimal constant starting with '&lt;<i>number</i>>'
has been detected.  </entry>
</row>
<row>
<entry colname="col1">42610</entry>
<entry colname="col2">All the arguments to the COALESCE/VALUE function cannot
be parameters. The function needs at least one argument that is not a parameter. </entry>
</row>
<row>
<entry colname="col1">42611</entry>
<entry colname="col2">The length, precision, or scale attribute for column,
or type mapping '&lt;<i>value</i>>' is not valid. </entry>
</row>
<row>
<entry colname="col1">42613</entry>
<entry colname="col2">Multiple or conflicting keywords involving the '&lt;<i>clause</i>>'
clause are present. </entry>
</row>
<row>
<entry colname="col1">42621</entry>
<entry colname="col2">A check constraint or generated column that is defined
with '&lt;<i>value</i>>' is invalid. </entry>
</row>
<row>
<entry colname="col1">42622</entry>
<entry colname="col2">The name '&lt;<i>name</i>>' is too long. The maximum
length is '&lt;<i>maximumLength</i>>'.  </entry>
</row>
<row>
<entry colname="col1">42734</entry>
<entry colname="col2">Name '&lt;<i>name</i>>' specified in context '&lt;<i>context</i>>'
is not unique. </entry>
</row>
<row>
<entry colname="col1">42802</entry>
<entry colname="col2">The number of values assigned is not the same as the
number of specified or implied columns. </entry>
</row>
<row>
<entry colname="col1">42803</entry>
<entry colname="col2">An expression containing the column '&lt;<i>columnName</i>>'
appears in the SELECT list and is not part of a GROUP BY clause.  </entry>
</row>
<row>
<entry colname="col1">42815</entry>
<entry colname="col2">The replacement value for '&lt;<i>value</i>>' is invalid. </entry>
</row>
<row>
<entry colname="col1">42815</entry>
<entry colname="col2">The data type, length or value of arguments '&lt;<i>firstArguement</i>>'
and '&lt;<i>secondArguement</i>>' is incompatible. </entry>
</row>
<row>
<entry colname="col1">42818</entry>
<entry colname="col2">Comparisons between '&lt;<i>value</i>>' and '&lt;<i>value</i>>'
are not supported.  </entry>
</row>
<row>
<entry colname="col1">42820</entry>
<entry colname="col2">The floating point literal '&lt;<i>string</i>>' contains
more than 30 characters.  </entry>
</row>
<row>
<entry colname="col1">42821</entry>
<entry colname="col2">Columns of type '&lt;<i>type</i>>' cannot hold values
of type '&lt;<i>type</i>>'.  </entry>
</row>
<row>
<entry colname="col1">42824</entry>
<entry colname="col2">An operand of LIKE is not a string, or the first operand
is not a column.</entry>
</row>
<row>
<entry colname="col1">42831</entry>
<entry colname="col2">'&lt;<i>columnName</i>>' cannot be a column of a primary
key or unique key because it can contain null values.</entry>
</row>
<row>
<entry colname="col1">42834</entry>
<entry colname="col2">SET NULL cannot be specified because FOREIGN KEY '&lt;<i>key</i>>'
 cannot contain null values.</entry>
</row>
<row>
<entry colname="col1">42837</entry>
<entry colname="col2">ALTER TABLE '&lt;<i>tableName</i>>' specified attributes
for column '&lt;<i>columnName</i>>' that are not compatible with the existing
column. </entry>
</row>
<row>
<entry colname="col1">42846</entry>
<entry colname="col2">Cannot convert types '&lt;<i>type</i>>' to '&lt;<i>type</i>>'. </entry>
</row>
<row>
<entry colname="col1">42877</entry>
<entry colname="col2">A qualified column name '&lt;<i>columnName</i>>' is
not allowed in the ORDER BY clause.  </entry>
</row>
<row>
<entry colname="col1">42884</entry>
<entry colname="col2">No authorized routine named '&lt;<i>routineName</i>>'
of type '&lt;<i>type</i>>' having compatible arguments was found. </entry>
</row>
<row>
<entry colname="col1">42886</entry>
<entry colname="col2">'&lt;<i>value</i>>' parameter '&lt;<i>value</i>>' requires
a parameter marker '&lt;<i>parameter</i>>'.  </entry>
</row>
<row>
<entry colname="col1">42894</entry>
<entry colname="col2">DEFAULT value or IDENTITY attribute value is not valid
for column '&lt;<i>columnName</i>>'.  </entry>
</row>
<row>
<entry colname="col1">428C1</entry>
<entry colname="col2">Only one identity column is allowed in a table.</entry>
</row>
<row>
<entry colname="col1">428EK</entry>
<entry colname="col2">The qualifier for a declared global temporary table
name must be SESSION.  </entry>
</row>
<row>
<entry colname="col1">42903</entry>
<entry colname="col2">Invalid use of an aggregate function. </entry>
</row>
<row>
<entry colname="col1">42908</entry>
<entry colname="col2">The CREATE VIEW statement does not include a column
list.  </entry>
</row>
<row>
<entry colname="col1">42915   </entry>
<entry colname="col2">Foreign Key '&lt;<i>key</i>>' is invalid because '&lt;<i>value</i>>'.</entry>
</row>
<row>
<entry colname="col1">42916</entry>
<entry colname="col2">Synonym '&lt;<i>synonym2</i>>' cannot be created for
'&lt;<i>synonym1</i>>' as it would result in a circular synonym chain.</entry>
</row>
<row>
<entry colname="col1">42939</entry>
<entry colname="col2">An object cannot be created with the schema name '&lt;<i>nome-do-esquema</i>>'.
  </entry>
</row>
<row>
<entry colname="col1">42962</entry>
<entry colname="col2">Long column type column or parameter '&lt;<i>columnName</i>>'
not permitted in declared global temporary tables or procedure definitions.
  </entry>
</row>
<row>
<entry colname="col1">42972</entry>
<entry colname="col2">An ON clause associated with a JOIN operator is not
valid.  </entry>
</row>
<row>
<entry colname="col1">42995</entry>
<entry colname="col2">The requested function does not apply to global temporary
tables. </entry>
</row>
<row>
<entry colname="col1">42X01  </entry>
<entry colname="col2">Syntax error: &lt;<i>error</i>>.                    </entry>
</row>
<row>
<entry colname="col1">42X02</entry>
<entry colname="col2">&lt;<i>value</i>>. </entry>
</row>
<row>
<entry colname="col1">42X03</entry>
<entry colname="col2">Column name '&lt;<i>columnName</i>>' is in more than
one table in the FROM list.  </entry>
</row>
<row>
<entry colname="col1">42X04</entry>
<entry colname="col2">Column '&lt;<i>columnName</i>>' is either not in any
table in the FROM list or appears within a join specification and is outside
the scope of the join specification or appears in a HAVING clause and is not
in the GROUP BY list.  If this is a CREATE or ALTER TABLE  statement then
'&lt;<i>columnName</i>>' is not a column in the target table. </entry>
</row>
<row>
<entry colname="col1">42X05</entry>
<entry colname="col2">Table '&lt;<i>tableName</i>>' does not exist. </entry>
</row>
<row>
<entry colname="col1">42X06</entry>
<entry colname="col2">Too many result columns specified for table '&lt;<i>tableName</i>>'.
 </entry>
</row>
<row>
<entry colname="col1">42X07</entry>
<entry colname="col2">Null is only allowed in a VALUES clause within an INSERT
statement. </entry>
</row>
<row>
<entry colname="col1">42X08</entry>
<entry colname="col2">The constructor for class '&lt;<i>className</i>>' cannot
be used as an external virtual table because the class does not implement
'&lt;<i>constructorName</i>>'.   </entry>
</row>
<row>
<entry colname="col1">42X09</entry>
<entry colname="col2">The table or alias name '&lt;<i>tableName</i>>' is used
more than once in the FROM list. </entry>
</row>
<row>
<entry colname="col1">42X10</entry>
<entry colname="col2"> '&lt;<i>tableName</i>>' is not an exposed table name
in the scope in which it appears.  </entry>
</row>
<row>
<entry colname="col1">42X12</entry>
<entry colname="col2">Column name '&lt;<i>tableName</i>>' appears more than
once in the CREATE TABLE statement. </entry>
</row>
<row>
<entry colname="col1">42X13</entry>
<entry colname="col2">Column name '&lt;<i>columnName</i>>' appears more than
once in the column list of an INSERT statement. </entry>
</row>
<row>
<entry colname="col1">42X14</entry>
<entry colname="col2">'&lt;<i>columnName</i>>' is not a column in table or
VTI '&lt;<i>value</i>>'. </entry>
</row>
<row>
<entry colname="col1">42X15</entry>
<entry colname="col2">Column name '&lt;<i>columnName</i>>' appears in a statement
without a FROM list.  </entry>
</row>
<row>
<entry colname="col1">42X16</entry>
<entry colname="col2">Column name '&lt;<i>columnName</i>>' appears multiple
times in the SET clause of an UPDATE statement. </entry>
</row>
<row>
<entry colname="col1">42X17</entry>
<entry colname="col2">In the Properties list of a FROM clause, the value '&lt;<i>value</i>>'
is not valid as a joinOrder specification. Only the values FIXED and UNFIXED
are valid.  </entry>
</row>
<row>
<entry colname="col1">42X19</entry>
<entry colname="col2">The WHERE or HAVING clause or CHECK CONSTRAINT definition
is a '&lt;<i>value</i>>' expression.  It must be a BOOLEAN expression.   </entry>
</row>
<row>
<entry colname="col1">42X23</entry>
<entry colname="col2">Cursor &lt;<i>cursorName</i>> is not updatable. </entry>
</row>
<row>
<entry colname="col1">42X25</entry>
<entry colname="col2">The '&lt;<i>functionName</i>>' function is not allowed
on the '&lt;<i>type</i>>' type.  </entry>
</row>
<row>
<entry colname="col1">42X26</entry>
<entry colname="col2">The class '&lt;<i>className</i>> for column '&lt;<i>columnName</i>>'
does not exist or is inaccessible. This can happen if the class is not public.
   </entry>
</row>
<row>
<entry colname="col1">42X28</entry>
<entry colname="col2">Delete table '&lt;<i>tableName</i>>' is not target of
cursor '&lt;<i>cursorName</i>>'.  </entry>
</row>
<row>
<entry colname="col1">42X29</entry>
<entry colname="col2">Update table '&lt;<i>tableName</i>>' is not the target
of cursor '&lt;<i>cursorName</i>>'.  </entry>
</row>
<row>
<entry colname="col1">42X30</entry>
<entry colname="col2">Cursor '&lt;<i>cursorName</i>>' not found. Verify that
autocommit is OFF.  </entry>
</row>
<row>
<entry colname="col1">42X31</entry>
<entry colname="col2">Column '&lt;<i>columnName</i>>' is not in the FOR UPDATE
list of cursor '&lt;<i>cursorName</i>>'.  </entry>
</row>
<row>
<entry colname="col1">42X32</entry>
<entry colname="col2">The number of columns in the derived column list must
match the number of columns in table '&lt;<i>tableName</i>>'.</entry>
</row>
<row>
<entry colname="col1">42X33</entry>
<entry colname="col2">The derived column list contains a duplicate column
name '&lt;<i>columnName</i>>'. </entry>
</row>
<row>
<entry colname="col1">42X34</entry>
<entry colname="col2">There is a ? parameter in the select list.  This is
not allowed. </entry>
</row>
<row>
<entry colname="col1">42X35 </entry>
<entry colname="col2">It is not allowed for both operands of '&lt;<i>value</i>>'
to be ? parameters. </entry>
</row>
<row>
<entry colname="col1">42X36</entry>
<entry colname="col2">The '&lt;<i>operator</i>>' operator is not allowed to
take a ? parameter as an operand.</entry>
</row>
<row>
<entry colname="col1">42X37</entry>
<entry colname="col2">The unary '&lt;<i>operator</i>>' operator is not allowed
on the '&lt;<i>type</i>>' type. </entry>
</row>
<row>
<entry colname="col1">42X38</entry>
<entry colname="col2">'SELECT *' only allowed in EXISTS and NOT EXISTS subqueries.
 </entry>
</row>
<row>
<entry colname="col1">42X39</entry>
<entry colname="col2">Subquery is only allowed to return a single column. </entry>
</row>
<row>
<entry colname="col1">42X40</entry>
<entry colname="col2">A NOT statement has an operand that is not boolean .
The operand of NOT must evaluate to TRUE, FALSE, or UNKNOWN. </entry>
</row>
<row>
<entry colname="col1">42X41</entry>
<entry colname="col2">In the Properties clause of a FROM list, the property
'&lt;<i>propertyName</i>>' is not valid (the property was being set to '&lt;<i>value</i>>'). </entry>
</row>
<row>
<entry colname="col1">42X42</entry>
<entry colname="col2">Correlation name not allowed for column '&lt;<i>columnName</i>>'
because it is part of the FOR UPDATE list.</entry>
</row>
<row>
<entry colname="col1">42X43</entry>
<entry colname="col2">The ResultSetMetaData returned for the class/object
'&lt;<i>className</i>>' was null. In order to use this class as an external
virtual table, the ResultSetMetaData cannot be null. </entry>
</row>
<row>
<entry colname="col1">42X44 </entry>
<entry colname="col2">Invalid length '&lt;<i>length</i>>' in column specification.
 </entry>
</row>
<row>
<entry colname="col1">42X45</entry>
<entry colname="col2">&lt;<i>type</i>> is an invalid type for argument number &lt;<i>value</i>>
of &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">42X48</entry>
<entry colname="col2">Value '&lt;<i>value</i>>' is not a valid precision for &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">42X49</entry>
<entry colname="col2">Value '&lt;<i>value</i>>' is not a valid integer literal.</entry>
</row>
<row>
<entry colname="col1">42X50</entry>
<entry colname="col2">No method was found that matched the method call &lt;<i>methodName</i>>.&lt;<i>value</i>>(&lt;<i>value</i>>),
tried all combinations of object and primitive types and any possible type
conversion for any  parameters the method call may have. The method might
exist but it is not public and/or static, or the parameter types are not method
invocation convertible.</entry>
</row>
<row>
<entry colname="col1">42X51</entry>
<entry colname="col2">The class '&lt;<i>className</i>>' does not exist or
is inaccessible. This can happen if the class is not public. </entry>
</row>
<row>
<entry colname="col1">42X52</entry>
<entry colname="col2">Calling method ('&lt;<i>methodName</i>>') using a receiver
of the Java primitive type '&lt;<i>type</i>>" is not allowed.  </entry>
</row>
<row>
<entry colname="col1">42X53   </entry>
<entry colname="col2">The LIKE predicate can only have 'CHAR' or 'VARCHAR'
operands. Type '&lt;<i>type</i>>' is not permitted. </entry>
</row>
<row>
<entry colname="col1">42X54</entry>
<entry colname="col2">The Java method '&lt;<i>methodName</i>>' has a ? as
a receiver.  This is not allowed.</entry>
</row>
<row>
<entry colname="col1">42X55</entry>
<entry colname="col2">Table name '&lt;<i>tableName</i>>' should be the same
as '&lt;<i>value</i>>'. </entry>
</row>
<row>
<entry colname="col1">42X56</entry>
<entry colname="col2">The number of columns in the view column list does not
match the number of columns in the underlying query expression in the view
definition for '&lt;<i>value</i>>'.  </entry>
</row>
<row>
<entry colname="col1">42X57</entry>
<entry colname="col2">The getColumnCount() for external virtual table '&lt;<i>tableName</i>>'
returned an invalid value '&lt;<i>value</i>>'.  Valid values are greater than
or equal to 1.  </entry>
</row>
<row>
<entry colname="col1">42X58</entry>
<entry colname="col2">The number of columns on the left and right sides of
the &lt;<i>tableName</i>> must be the same.</entry>
</row>
<row>
<entry colname="col1">42X59</entry>
<entry colname="col2">The number of columns in each VALUES constructor must
be the same.  </entry>
</row>
<row>
<entry colname="col1">42X60</entry>
<entry colname="col2">Invalid value '&lt;<i>value</i>>' for insertMode property
specified for table '&lt;<i>tableName</i>>'.  </entry>
</row>
<row>
<entry colname="col1">42X61 </entry>
<entry colname="col2">Types '&lt;<i>type</i>>' and '&lt;<i>type</i>>' are
not &lt;<i>value</i>> compatible. </entry>
</row>
<row>
<entry colname="col1">42X62</entry>
<entry colname="col2">'&lt;<i>value</i>>' is not allowed in the '&lt;<i>nome-do-esquema</i>>'
schema. </entry>
</row>
<row>
<entry colname="col1">42X63</entry>
<entry colname="col2">The USING clause did not return any results. No parameters
can be set. </entry>
</row>
<row>
<entry colname="col1">42X64</entry>
<entry colname="col2">In the Properties list, the invalid value '&lt;<i>value</i>>'
was specified for the useStatistics property. The only valid values are TRUE
or FALSE . </entry>
</row>
<row>
<entry colname="col1">42X65</entry>
<entry colname="col2">Index '&lt;<i>index</i>>' does not exist.  </entry>
</row>
<row>
<entry colname="col1">42X66</entry>
<entry colname="col2">Column name '&lt;<i>columnName</i>>' appears more than
once in the CREATE INDEX statement.  </entry>
</row>
<row>
<entry colname="col1">42X68</entry>
<entry colname="col2">No field '&lt;<i>fieldName</i>>' was found belonging
to class '&lt;<i>className</i>>'.  The field might exist, but it is not public,
or the class does not exist or is not public. </entry>
</row>
<row>
<entry colname="col1">42X69</entry>
<entry colname="col2">It is not allowed to reference a field ('&lt;<i>fieldName</i>>')
using a referencing expression of the Java primitive type '&lt;<i>type</i>>'. </entry>
</row>
<row>
<entry colname="col1">42X72</entry>
<entry colname="col2">No static field '&lt;<i>fieldName</i>>' was found belonging
to class '&lt;<i>className</i>>'.  The field might exist, but it is not public
and/or static, or the class does not exist or the class is not public.  </entry>
</row>
<row>
<entry colname="col1">42X73</entry>
<entry colname="col2">Method resolution for signature &lt;<i>value</i>>.&lt;<i>value</i>>(&lt;<i>value</i>>)
was ambiguous. (No single maximally specific method.)  </entry>
</row>
<row>
<entry colname="col1">42X74</entry>
<entry colname="col2">Invalid CALL statement syntax.</entry>
</row>
<row>
<entry colname="col1">42X75</entry>
<entry colname="col2">No constructor was found with the signature &lt;<i>value</i>>(&lt;<i>value</i>>).
 It may be that the parameter types are not method invocation convertible.</entry>
</row>
<row>
<entry colname="col1">42X76</entry>
<entry colname="col2">At least one column, '&lt;<i>columnName</i>>', in the
primary key being added is nullable. All columns in a primary key must be
non-nullable.  </entry>
</row>
<row>
<entry colname="col1">42X77</entry>
<entry colname="col2">Column position '&lt;<i>columnPosition</i>>' is out
of range for the query expression.  </entry>
</row>
<row>
<entry colname="col1">42X78</entry>
<entry colname="col2">Column '&lt;<i>columnName</i>>' is not in the result
of the query expression. </entry>
</row>
<row>
<entry colname="col1">42X79</entry>
<entry colname="col2">Column name '&lt;<i>columnName</i>>' appears more than
once in the result of the query expression. </entry>
</row>
<row>
<entry colname="col1">42X80</entry>
<entry colname="col2">VALUES clause must contain at least one element. Empty
elements are not allowed. </entry>
</row>
<row>
<entry colname="col1">42X82</entry>
<entry colname="col2">The USING clause returned more than one row. Only single-row
ResultSets are permissible. </entry>
</row>
<row>
<entry colname="col1">42X83</entry>
<entry colname="col2">The constraints on column '&lt;<i>columnName</i>>' require
that it be both nullable and not nullable.  </entry>
</row>
<row>
<entry colname="col1">42X84</entry>
<entry colname="col2">Index '&lt;<i>index</i>>' was created to enforce constraint
'&lt;<i>constraintName</i>>'.  It can only be dropped by dropping the constraint. </entry>
</row>
<row>
<entry colname="col1">42X85</entry>
<entry colname="col2">Constraint '&lt;<i>constraintName</i>>' is required
to be in the same schema as table '&lt;<i>tableName</i>>'.  </entry>
</row>
<row>
<entry colname="col1">42X86</entry>
<entry colname="col2">ALTER TABLE failed. There is no constraint '&lt;<i>constraintName</i>>'
on table '&lt;<i>tableName</i>>'.   </entry>
</row>
<row>
<entry colname="col1">42X87</entry>
<entry colname="col2">At least one result expression (THEN or ELSE) of the
'&lt;<i>expressão</i>>' expression must not be a '?'.  </entry>
</row>
<row>
<entry colname="col1">42X88</entry>
<entry colname="col2">A conditional has a non-boolean operand. The operand
of a conditional must evaluate to TRUE, FALSE, or UNKNOWN.  </entry>
</row>
<row>
<entry colname="col1">42X89</entry>
<entry colname="col2">Types '&lt;<i>type</i>>' and '&lt;<i>type</i>>' are
not type compatible. Neither type is assignable to the other type.  </entry>
</row>
<row>
<entry colname="col1">42X90</entry>
<entry colname="col2">More than one primary key constraint specified for table
'&lt;<i>tableName</i>>'.  </entry>
</row>
<row>
<entry colname="col1">42X91</entry>
<entry colname="col2">Constraint name '&lt;<i>constraintName</i>>' appears
more than once in the CREATE TABLE statement. </entry>
</row>
<row>
<entry colname="col1">42X92</entry>
<entry colname="col2">Column name '&lt;<i>columnName</i>>' appears more than
once in a constraint's column list.</entry>
</row>
<row>
<entry colname="col1">42X93</entry>
<entry colname="col2">Table '&lt;<i>tableName</i>>' contains a constraint
definition with column '&lt;<i>columnName</i>>' which is not in the table. </entry>
</row>
<row>
<entry colname="col1">42X94</entry>
<entry colname="col2">&lt;<i>value</i>> '&lt;<i>value</i>>' does not exist.</entry>
</row>
<row>
<entry colname="col1">42X96</entry>
<entry colname="col2">The database classpath contains an unknown jar file
'&lt;<i>fileName</i>>'. </entry>
</row>
<row>
<entry colname="col1">42X98</entry>
<entry colname="col2">Parameters are not allowed in a VIEW definition. </entry>
</row>
<row>
<entry colname="col1">42Y00</entry>
<entry colname="col2">Class '&lt;<i>className</i>>' does not implement org.apache.derby.iapi.db.AggregateDefinition
and thus cannot be used as an aggregate expression.</entry>
</row>
<row>
<entry colname="col1">42Y01</entry>
<entry colname="col2">Constraint '&lt;<i>constraintName</i>>' is invalid.
  </entry>
</row>
<row>
<entry colname="col1">42Y03</entry>
<entry colname="col2">'&lt;<i>statement</i>>' is not recognized as a function
or procedure. </entry>
</row>
<row>
<entry colname="col1">42Y04</entry>
<entry colname="col2">Cannot create a procedure or function with EXTERNAL
NAME '&lt;<i>name</i>>' because it is not a list separated by periods. The
expected format is &lt;<i>fulljavapath</i>>.&lt;<i>method name</i>>.</entry>
</row>
<row>
<entry colname="col1">42Y05</entry>
<entry colname="col2">There is no Foreign Key named '&lt;<i>key</i>>'. </entry>
</row>
<row>
<entry colname="col1">42Y07</entry>
<entry colname="col2">Schema '&lt;<i>nome-do-esquema</i>>' does not exist.</entry>
</row>
<row>
<entry colname="col1">42Y08</entry>
<entry colname="col2">Foreign key constraints are not allowed on system tables. </entry>
</row>
<row>
<entry colname="col1">42Y09</entry>
<entry colname="col2">Void methods are only allowed within a CALL statement. </entry>
</row>
<row>
<entry colname="col1"> 42Y10</entry>
<entry colname="col2">A table constructor that is not in an INSERT statement
has all ? parameters in one of its columns. For each column, at least one
of the rows must have a non-parameter.</entry>
</row>
<row>
<entry colname="col1">42Y11</entry>
<entry colname="col2">A join specification is required with the '&lt;<i>clauseName</i>>'
clause. </entry>
</row>
<row>
<entry colname="col1">42Y12</entry>
<entry colname="col2">The ON clause of a JOIN is a '&lt;<i>expressionType</i>>'
expression.  It must be a BOOLEAN expression. </entry>
</row>
<row>
<entry colname="col1">42Y13</entry>
<entry colname="col2">Column name '&lt;<i>columnName</i>>' appears more than
once in the CREATE VIEW statement.</entry>
</row>
<row>
<entry colname="col1">42Y16</entry>
<entry colname="col2">No public static method '&lt;<i>methodName</i>>' was
found in class '&lt;<i>className</i>>'. The method might exist, but it is
not public, or it is not static. </entry>
</row>
<row>
<entry colname="col1">42Y19</entry>
<entry colname="col2">'&lt;<i>columnName</i>>' appears multiple times in the
GROUP BY list. Columns in the GROUP BY list must be unambiguous.  </entry>
</row>
<row>
<entry colname="col1">42Y22</entry>
<entry colname="col2">Aggregate &lt;<i>aggregateType</i>> cannot operate on
type &lt;<i>type</i>>. </entry>
</row>
<row>
<entry colname="col1">42Y23</entry>
<entry colname="col2">Incorrect JDBC type info returned for column &lt;<i>columnName</i>>.
  </entry>
</row>
<row>
<entry colname="col1">42Y24</entry>
<entry colname="col2">View '&lt;<i>viewName</i>>' is not updatable. (Views
are currently not updatable.) </entry>
</row>
<row>
<entry colname="col1">42Y25</entry>
<entry colname="col2">'&lt;<i>tableName</i>>' is a system table.  Users are
not allowed to modify the contents of this table. </entry>
</row>
<row>
<entry colname="col1">42Y27</entry>
<entry colname="col2">Parameters are not allowed in the trigger action.   </entry>
</row>
<row>
<entry colname="col1">42Y29</entry>
<entry colname="col2">The SELECT list of a non-grouped query contains at least
one invalid expression.  When the SELECT list contains at least one aggregate
then all entries must be valid aggregate expressions.</entry>
</row>
<row>
<entry colname="col1">42Y30</entry>
<entry colname="col2">The SELECT list of a grouped query contains at least
one invalid expression.  If a SELECT list has a GROUP BY, the list may only
contain grouping columns and valid aggregate expressions.  </entry>
</row>
<row>
<entry colname="col1">42Y32</entry>
<entry colname="col2">Aggregator class '&lt;<i>className</i>> aggregate '&lt;<i>aggregateName</i>>'
on type &lt;<i>type</i>> does not implement com.ibm.db2j.aggregates.Aggregator.
  </entry>
</row>
<row>
<entry colname="col1">42Y33</entry>
<entry colname="col2">Aggregate &lt;<i>aggregateName</i>> contains one or
more aggregates.   </entry>
</row>
<row>
<entry colname="col1">42Y34</entry>
<entry colname="col2">Column name '&lt;<i>columnName</i>>' matches more than
one result column in table '&lt;<i>tableName</i>>'.  </entry>
</row>
<row>
<entry colname="col1">42Y35</entry>
<entry colname="col2">Column reference '&lt;<i>reference</i>>' is invalid.
When the SELECT list contains at least one aggregate then all entries must
be valid aggregate expressions.   </entry>
</row>
<row>
<entry colname="col1">42Y36</entry>
<entry colname="col2">Column reference '&lt;<i>value</i>>' is invalid.  For
a SELECT list with a GROUP BY, the list may only contain grouping columns
and valid aggregate expressions.  </entry>
</row>
<row>
<entry colname="col1">42Y37</entry>
<entry colname="col2">'&lt;<i>value</i>>' is a Java primitive and cannot be
used with this operator.   </entry>
</row>
<row>
<entry colname="col1">42Y38</entry>
<entry colname="col2"><codeblock>insertMode = replace</codeblock> is not permitted
on an insert where the target table, '&lt;<i>tableName</i>>', is referenced
in the SELECT.</entry>
</row>
<row>
<entry colname="col1">42Y39</entry>
<entry colname="col2">'&lt;<i>value</i>>' may not appear in a CHECK CONSTRAINT
definition because it may return non-deterministic results.  </entry>
</row>
<row>
<entry colname="col1">42Y40</entry>
<entry colname="col2">'&lt;<i>value</i>>' appears multiple times in the UPDATE
OF column list for trigger '&lt;<i>triggerName</i>>'. </entry>
</row>
<row>
<entry colname="col1">42Y41</entry>
<entry colname="col2">'&lt;<i>value</i>>' cannot be directly invoked via EXECUTE
STATEMENT because it is part of a trigger.    </entry>
</row>
<row>
<entry colname="col1">42Y42</entry>
<entry colname="col2">Scale '&lt;<i>scaleValue</i>>' is not a valid scale
for a &lt;<i>value</i>>. </entry>
</row>
<row>
<entry colname="col1">42Y43</entry>
<entry colname="col2">Scale '&lt;<i>scaleValue</i>>' is not a valid scale
with precision of '&lt;<i>precision</i>>'.  </entry>
</row>
<row>
<entry colname="col1">42Y44</entry>
<entry colname="col2">Invalid key '&lt;<i>key</i>>' specified in the Properties
list of a FROM list. The case-sensitive keys that are currently supported
are '&lt;<i>key</i>>'. </entry>
</row>
<row>
<entry colname="col1">42Y45</entry>
<entry colname="col2">VTI '&lt;<i>value</i>>' cannot be bound because it is
a special trigger VTI and this statement is not part of a trigger action or
WHEN clause.</entry>
</row>
<row>
<entry colname="col1">42Y46</entry>
<entry colname="col2">Invalid Properties list in FROM list.  There is no index
'&lt;<i>index</i>>' on table '&lt;<i>tableName</i>>'.</entry>
</row>
<row>
<entry colname="col1">42Y48</entry>
<entry colname="col2">Invalid Properties list in FROM list.  Either there
is no named constraint '&lt;<i>constraintName</i>>' on table '&lt;<i>tableName</i>>'
or the constraint does not have a backing index. </entry>
</row>
<row>
<entry colname="col1">42Y49</entry>
<entry colname="col2">Multiple values specified for property key '&lt;<i>key</i>>'.
 </entry>
</row>
<row>
<entry colname="col1">42Y50</entry>
<entry colname="col2">Properties list for table '&lt;<i>tableName</i>>' may
contain values for index or for constraint but not both.     </entry>
</row>
<row>
<entry colname="col1">42Y55</entry>
<entry colname="col2">'&lt;<i>value</i>>' cannot be performed on '&lt;<i>value</i>>'
because it does not exist. </entry>
</row>
<row>
<entry colname="col1">42Y56</entry>
<entry colname="col2">Invalid join strategy '&lt;<i>strategyValue</i>>' specified
in Properties list on table '&lt;<i>tableName</i>>'. The currently supported
values for a join strategy are: <codeblock>hash</codeblock> and <codeblock>nestedloop</codeblock>.</entry>
</row>
<row>
<entry colname="col1">42Y58</entry>
<entry colname="col2">NumberFormatException occurred when converting value
'&lt;<i>value</i>>' for optimizer override '&lt;<i>value</i>>'. </entry>
</row>
<row>
<entry colname="col1">42Y59</entry>
<entry colname="col2">Invalid value, '&lt;<i>value</i>>', specified for hashInitialCapacity
override. Value must be greater than 0. </entry>
</row>
<row>
<entry colname="col1">42Y60</entry>
<entry colname="col2">Invalid value, '&lt;<i>value</i>>', specified for hashLoadFactor
override. Value must be greater than 0.0 and less than or equal to 1.0. </entry>
</row>
<row>
<entry colname="col1">42Y61</entry>
<entry colname="col2">Invalid value, '&lt;<i>value</i>>' specified for hashMaxCapacity
override. Value must be greater than 0. </entry>
</row>
<row>
<entry colname="col1">42Y62</entry>
<entry colname="col2">'&lt;<i>value</i>>' is not allowed on '&lt;<i>value</i>>'
because it is a view. </entry>
</row>
<row>
<entry colname="col1">42Y63</entry>
<entry colname="col2">Hash join requires an optimizable equijoin predicate
on a column in the selected index or heap.  An optimizable equijoin predicate
does not exist on any column in table or index '&lt;<i>index</i>>'. Use the
'index' optimizer override to specify such an index or the heap on table '&lt;<i>tableName</i>>'. </entry>
</row>
<row>
<entry colname="col1">42Y64</entry>
<entry colname="col2">bulkFetch value of '&lt;<i>value</i>>' is invalid. The
minimum value for bulkFetch is 1.  </entry>
</row>
<row>
<entry colname="col1">42Y65</entry>
<entry colname="col2">bulkFetch is not permitted on '&lt;<i>joinType</i>>'
joins. </entry>
</row>
<row>
<entry colname="col1">42Y66</entry>
<entry colname="col2">bulkFetch is not permitted on updatable cursors.  </entry>
</row>
<row>
<entry colname="col1">42Y67</entry>
<entry colname="col2">Schema '&lt;<i>nome-do-esquema</i>>' cannot be dropped.</entry>
</row>
<row>
<entry colname="col1">42Y69 </entry>
<entry colname="col2">No valid execution plan was found for this statement.
This may have one of two causes: either you specified a hash join strategy
when hash join is not allowed (no optimizable equijoin) or you are attempting
to join two external virtual tables, each of which references the other, and
so the statement cannot be evaluated.  </entry>
</row>
<row>
<entry colname="col1"> 42Y70</entry>
<entry colname="col2">The user specified an illegal join order. This could
be caused by a join column from an inner table being passed as a parameter
to an external virtual table.  </entry>
</row>
<row>
<entry colname="col1">42Y71</entry>
<entry colname="col2">System function or procedure '&lt;<i>procedureName</i>>'
cannot be dropped.</entry>
</row>
<row>
<entry colname="col1">42Y82</entry>
<entry colname="col2">System generated stored prepared statement '&lt;<i>statement</i>>'
that cannot be dropped using DROP STATEMENT. It is part of a trigger. </entry>
</row>
<row>
<entry colname="col1"> 42Y83</entry>
<entry colname="col2">An untyped null is not permitted as an argument to aggregate &lt;<i>aggregateName</i>>.
 Please cast the null to a suitable type. </entry>
</row>
<row>
<entry colname="col1">42Y84</entry>
<entry colname="col2">'&lt;<i>value</i>>' may not appear in a DEFAULT definition.
  </entry>
</row>
<row>
<entry colname="col1">42Y85</entry>
<entry colname="col2">The DEFAULT keyword is only allowed in a VALUES clause
when the VALUES clause appears within an INSERT statement. </entry>
</row>
<row>
<entry colname="col1">42Y90</entry>
<entry colname="col2">FOR UPDATE is not permitted in this type of statement.
 </entry>
</row>
<row>
<entry colname="col1">42Y91</entry>
<entry colname="col2">The USING clause is not permitted in an EXECUTE STATEMENT
for a trigger action.  </entry>
</row>
<row>
<entry colname="col1">42Y92</entry>
<entry colname="col2">&lt;<i>triggerName</i>> triggers may only reference &lt;<i>value</i>>
transition variables/tables. </entry>
</row>
<row>
<entry colname="col1">42Y93</entry>
<entry colname="col2">Illegal REFERENCING clause: only one name is permitted
for each type of transition variable/table.  </entry>
</row>
<row>
<entry colname="col1">42Y94</entry>
<entry colname="col2">An AND or OR has a non-boolean operand. The operands
of AND and OR must evaluate to TRUE, FALSE, or UNKNOWN.   </entry>
</row>
<row>
<entry colname="col1">42Y95</entry>
<entry colname="col2">The '&lt;<i>operatorName</i>>' operator with a left
operand type of '&lt;<i>operandType</i>>' and a right operand type of '&lt;<i>operandType</i>>'
is not supported.   </entry>
</row>
<row>
<entry colname="col1">42Y97</entry>
<entry colname="col2">Invalid escape character at line '&lt;<i>lineNumber</i>>',
column '&lt;<i>columnName</i>>'.  </entry>
</row>
<row>
<entry colname="col1">42Z02</entry>
<entry colname="col2">Multiple DISTINCT aggregates are not supported at this
time.   </entry>
</row>
<row>
<entry colname="col1">42Z07</entry>
<entry colname="col2">Aggregates are not permitted in the ON clause.  </entry>
</row>
<row>
<entry colname="col1">42Z08</entry>
<entry colname="col2">Bulk insert replace is not permitted on '&lt;<i>value</i>>'
because it has an enabled trigger (&lt;<i>value</i>>). </entry>
</row>
<row>
<entry colname="col1">42Z15</entry>
<entry colname="col2">Invalid type specified for column '&lt;<i>columnName</i>>'.
The type of a column may not be changed.  </entry>
</row>
<row>
<entry colname="col1">42Z16</entry>
<entry colname="col2">Only columns of type VARCHAR may have their length altered. </entry>
</row>
<row>
<entry colname="col1">42Z17</entry>
<entry colname="col2">Invalid length specified for column '&lt;<i>columnName</i>>'.
Length must be greater than the current column length. </entry>
</row>
<row>
<entry colname="col1">42Z18</entry>
<entry colname="col2">Column '&lt;<i>columnName</i>>' is part of a foreign
key constraint '&lt;<i>constraintName</i>>'. To alter the length of this column,
you should drop the constraint first, perform the ALTER TABLE, and then recreate
the constraint.</entry>
</row>
<row>
<entry colname="col1">42Z19</entry>
<entry colname="col2">Column '&lt;<i>columnName</i>>' is being referenced
by at least one foreign key constraint '&lt;<i>constraintName</i>>'. To alter
the length of this column, you should drop referencing constraints, perform
the ALTER TABLE, and then recreate the constraints. </entry>
</row>
<row>
<entry colname="col1">42Z20</entry>
<entry colname="col2">Column '&lt;<i>columnName</i>>' cannot be made nullable.
It is part of a primary key, which cannot have any nullable columns.  </entry>
</row>
<row>
<entry colname="col1">42Z21</entry>
<entry colname="col2">Invalid increment specified for identity for column
'&lt;<i>columnName</i>>'. Increment cannot be zero.  </entry>
</row>
<row>
<entry colname="col1">42Z22</entry>
<entry colname="col2">Invalid type specified for identity column '&lt;<i>columnName</i>>'.
The only valid types for identity columns are BIGINT, INT and SMALLINT.  </entry>
</row>
<row>
<entry colname="col1">42Z23</entry>
<entry colname="col2">Attempt to modify an identity column '&lt;<i>columnName</i>>'. </entry>
</row>
<row>
<entry colname="col1">42Z24</entry>
<entry colname="col2">Overflow occurred in identity value for column '&lt;<i>columnName</i>>'
in table '&lt;<i>tableName</i>>'. </entry>
</row>
<row>
<entry colname="col1"> 42Z25</entry>
<entry colname="col2">INTERNAL ERROR identity counter. Update was called without
 arguments with <codeblock>current value = NULL</codeblock>.      </entry>
</row>
<row>
<entry colname="col1">42Z26</entry>
<entry colname="col2">A column, '&lt;<i>columnName</i>>', with an identity
default cannot be made nullable.  </entry>
</row>
<row>
<entry colname="col1">42Z27</entry>
<entry colname="col2">A nullable column, '&lt;<i>columnName</i>>', cannot
be modified to have identity default. </entry>
</row>
<row>
<entry colname="col1">42Z50</entry>
<entry colname="col2">INTERNAL ERROR: Unable to generate code for &lt;<i>value</i>>. </entry>
</row>
<row>
<entry colname="col1">42Z53</entry>
<entry colname="col2">INTERNAL ERROR: Do not know what type of activation
to generate for node choice &lt;<i>value</i>>. </entry>
</row>
<row>
<entry colname="col1">42Z90</entry>
<entry colname="col2">Class '&lt;<i>className</i>>' does not return an updatable
ResultSet. </entry>
</row>
<row>
<entry colname="col1">42Z91</entry>
<entry colname="col2">subquery</entry>
</row>
<row>
<entry colname="col1">42Z92</entry>
<entry colname="col2">repeatable read</entry>
</row>
<row>
<entry colname="col1">42Z93</entry>
<entry colname="col2">Constraints '&lt;<i>constraintName</i>>' and '&lt;<i>constraintName</i>>'
have the same set of columns, which is not allowed. </entry>
</row>
<row>
<entry colname="col1">42Z97</entry>
<entry colname="col2">Renaming column '&lt;<i>columnName</i>>' will cause
check constraint '&lt;<i>constraintName</i>>' to break.</entry>
</row>
<row>
<entry colname="col1">42Z99</entry>
<entry colname="col2">String or Hex literal cannot exceed 64K.</entry>
</row>
<row>
<entry colname="col1">42Z9A</entry>
<entry colname="col2">read uncommitted</entry>
</row>
<row>
<entry colname="col1">42Z9B</entry>
<entry colname="col2">The external virtual table interface does not support
BLOB or CLOB columns. '&lt;<i>value</i>>' column '&lt;<i>value</i>>'. </entry>
</row>
<row>
<entry colname="col1">42Z9D</entry>
<entry colname="col2">'&lt;<i>statement</i>>' statements are not allowed in
'&lt;<i>triggerName</i>>' triggers.  </entry>
</row>
<row>
<entry colname="col1">42Z9E</entry>
<entry colname="col2">Constraint '&lt;<i>constraintName</i>>' is not a &lt;<i>value</i>>
constraint.  </entry>
</row>
<row>
<entry colname="col1">42Z9F</entry>
<entry colname="col2">Too many indexes (&lt;<i>index</i>>) on the table &lt;<i>tableName</i>>.
The limit is &lt;<i>number</i>>.  </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XOX: Exceção de execução</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="129*"/>
<colspec colname="col2"
colwidth="673*"/>
<thead>
<row valign="bottom">
<entry colname="col1">SQLState</entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">X0X02</entry>
<entry colname="col2">Table '&lt;<i>tableName</i>>' cannot be locked in '&lt;<i>mode</i>>'
mode.  </entry>
</row>
<row>
<entry colname="col1">X0X03</entry>
<entry colname="col2">Invalid transaction state - held cursor requires same
isolation level    </entry>
</row>
<row>
<entry colname="col1">X0X05</entry>
<entry colname="col2">Table '&lt;<i>tableName</i>>' does not exist.   </entry>
</row>
<row>
<entry colname="col1">X0X07</entry>
<entry colname="col2">Cannot drop jar file '&lt;<i>fileName</i>>' because
its on your db2j.database.classpath '&lt;<i>classpath</i>>'.   </entry>
</row>
<row>
<entry colname="col1">X0X0E</entry>
<entry colname="col2">The column position '&lt;<i>columnPosition</i>>' listed
in the auto-generated column selection array was not found in the insert table. </entry>
</row>
<row>
<entry colname="col1">X0X0F</entry>
<entry colname="col2">Column name '&lt;<i>columnName</i>>' listed in auto-generated
column selection array not found in the insert table. </entry>
</row>
<row>
<entry colname="col1">X0X10</entry>
<entry colname="col2">The USING clause returned more than one row; only single-row
ResultSets are permissible.</entry>
</row>
<row>
<entry colname="col1">X0X11</entry>
<entry colname="col2">The USING clause did not return any results so no parameters
can be set.   </entry>
</row>
<row>
<entry colname="col1">X0X13 </entry>
<entry colname="col2">Jar file '&lt;<i>fileName</i>>' does not exist in schema
'&lt;<i>nome-do-esquema</i>>'. </entry>
</row>
<row>
<entry colname="col1">X0X14</entry>
<entry colname="col2">Binding directly to an XML value is not allowed; try
using XMLPARSE.</entry>
</row>
<row>
<entry colname="col1">X0X15</entry>
<entry colname="col2">XML values are not allowed in top-level result sets;
try using XMLSERIALIZE.</entry>
</row>
<row>
<entry colname="col1">X0X16</entry>
<entry colname="col2">XML syntax error; missing keyword(s): '&lt;<i>keywords</i>>'.</entry>
</row>
<row>
<entry colname="col1">X0X17</entry>
<entry colname="col2">Invalid target type for XMLSERIALIZE: '&lt;<i>value</i>>'.</entry>
</row>
<row>
<entry colname="col1">X0X18</entry>
<entry colname="col2">XML feature not supported: '&lt;<i>featureName</i>>'.</entry>
</row>
<row>
<entry colname="col1">X0X57</entry>
<entry colname="col2">An attempt was made to put a Java value of type '&lt;<i>type</i>>'
into a SQL value, but there is no corresponding SQL type.  The Java value
is probably the result of a method call or field access.    </entry>
</row>
<row>
<entry colname="col1">X0X60</entry>
<entry colname="col2">A cursor with name '&lt;<i>cursorName</i>>' already
exists.   </entry>
</row>
<row>
<entry colname="col1">X0X61</entry>
<entry colname="col2">The values for column '&lt;<i>value</i>>' in index '&lt;<i>value</i>>'
and table '&lt;<i>value</i>>.&lt;<i>value</i>>' do not match for row location &lt;<i>value</i>>.
 The value in the index is '&lt;<i>value</i>>', while the value in the base
table is '&lt;<i>value</i>>'.  The full index key, including the row location,
is '&lt;<i>value</i>>'.  The suggested corrective action is to recreate the
index. </entry>
</row>
<row>
<entry colname="col1">X0X62</entry>
<entry colname="col2">Inconsistency found between table '&lt;<i>tableName</i>>'
and index '&lt;<i>index</i>>'. Error when trying to retrieve row location
'&lt;<i>rowLocation</i>>' from the table.  The full index key, including the
row location, is '&lt;<i>index</i>>'. The suggested corrective action is to
recreate the index.       </entry>
</row>
<row>
<entry colname="col1">X0X63</entry>
<entry colname="col2">Got IOException '&lt;<i>value</i>>'.</entry>
</row>
<row>
<entry colname="col1">X0X67</entry>
<entry colname="col2">Columns of type '&lt;<i>type</i>>' may not be used in
CREATE INDEX, ORDER BY, GROUP BY, UNION, INTERSECT, EXCEPT or DISTINCT statements
because comparisons are not supported for that type.</entry>
</row>
<row>
<entry colname="col1">X0X81</entry>
<entry colname="col2">&lt;<i>value</i>> '&lt;<i>value</i>>' does not exist.
   </entry>
</row>
<row>
<entry colname="col1">X0X85</entry>
<entry colname="col2">Index '&lt;<i>index</i>>' was not created because '&lt;<i>type</i>>'
is not a valid index type.   </entry>
</row>
<row>
<entry colname="col1">X0X86</entry>
<entry colname="col2">0 is an invalid parameter value for <codeblock>ResultSet.absolute(int row)</codeblock>.
    </entry>
</row>
<row>
<entry colname="col1">X0X87</entry>
<entry colname="col2"><codeblock>ResultSet.relative(int row)</codeblock> cannot
be called when the cursor is not positioned on a row.   </entry>
</row>
<row>
<entry colname="col1">X0X95</entry>
<entry colname="col2">Operation '&lt;<i>operation</i>>' cannot be performed
on object '&lt;<i>object</i>>' because there is an open ResultSet dependent
on that object.         </entry>
</row>
<row>
<entry colname="col1">X0X99</entry>
<entry colname="col2">Index '&lt;<i>index</i>>' does not exist.  </entry>
</row>
<row>
<entry colname="col1">X0XML</entry>
<entry colname="col2">Encountered unexpected error while processing XML; see
next exception for details.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe X0Y: Exceções de execução</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">X0Y16</entry>
<entry colname="col2">'&lt;<i>value</i>>' is not a view.  If it is a table,
then use DROP TABLE instead. </entry>
</row>
<row>
<entry colname="col1">X0Y23</entry>
<entry colname="col2">Operation '&lt;<i>operation</i>>' cannot be performed
on object '&lt;<i>object</i>>' because VIEW '&lt;<i>viewName</i>>' is dependent
on that object. </entry>
</row>
<row>
<entry colname="col1">X0Y24</entry>
<entry colname="col2">Operation '&lt;<i>operation</i>>' cannot be performed
on object '&lt;<i>object</i>>' because STATEMENT '&lt;<i>statement</i>>' is
dependent on that object.</entry>
</row>
<row>
<entry colname="col1">X0Y25</entry>
<entry colname="col2">Operation '&lt;<i>value</i>>' cannot be performed on
object '&lt;<i>value</i>>' because &lt;<i>value</i>> '&lt;<i>value</i>>' is
dependent on that object.  </entry>
</row>
<row>
<entry colname="col1">X0Y26     </entry>
<entry colname="col2">Index '&lt;<i>index</i>>' is required to be in the same
schema as table '&lt;<i>tableName</i>>'.   </entry>
</row>
<row>
<entry colname="col1">X0Y28</entry>
<entry colname="col2">Index '&lt;<i>index</i>>' cannot be created on system
table '&lt;<i>tableName</i>>'.  Users cannot create indexes on system tables. </entry>
</row>
<row>
<entry colname="col1">X0Y32</entry>
<entry colname="col2">&lt;<i>value</i>> '&lt;<i>value</i>>' already exists
in &lt;<i>value</i>> '&lt;<i>value</i>>'.  </entry>
</row>
<row>
<entry colname="col1">X0Y38     </entry>
<entry colname="col2">Cannot create index '&lt;<i>index</i>>' because table
'&lt;<i>tableName</i>>' does not exist.  </entry>
</row>
<row>
<entry colname="col1">X0Y41</entry>
<entry colname="col2">Constraint '&lt;<i>constraintName</i>>' is invalid because
the referenced table &lt;<i>tableName</i>> has no primary key.  Either add
a primary key to &lt;<i>tableName</i>> or explicitly specify the columns of
a unique constraint that this foreign key references.   </entry>
</row>
<row>
<entry colname="col1">X0Y42</entry>
<entry colname="col2">Constraint '&lt;<i>constraintName</i>>' is invalid:
the types of the foreign key columns do not match the types of the referenced
columns</entry>
</row>
<row>
<entry colname="col1">X0Y43</entry>
<entry colname="col2">Constraint '&lt;<i>value</i>>' is invalid: the number
of columns in &lt;<i>value</i>> (&lt;<i>value</i>>) does not match the number
of columns in the referenced key (&lt;<i>value</i>>).   </entry>
</row>
<row>
<entry colname="col1">X0Y44    </entry>
<entry colname="col2">Constraint '&lt;<i>constraintName</i>>' is invalid:
there is no unique or primary key constraint on table '&lt;<i>tableName</i>>'
that matches the number and types of the columns in the foreign key.
   </entry>
</row>
<row>
<entry colname="col1">X0Y45 </entry>
<entry colname="col2">Foreign key constraint '&lt;<i>constraintName</i>>'
cannot be added to or enabled on table &lt;<i>tableName</i>> because one or
more foreign keys do not have matching referenced keys. </entry>
</row>
<row>
<entry colname="col1">X0Y46</entry>
<entry colname="col2">Constraint '&lt;<i>constraintName</i>>' is invalid:
referenced table &lt;<i>tableName</i>> does not exist.    </entry>
</row>
<row>
<entry colname="col1">X0Y54</entry>
<entry colname="col2">Schema '&lt;<i>nome-do-esquema</i>>' cannot be dropped because
it is not empty.     </entry>
</row>
<row>
<entry colname="col1">X0Y55     </entry>
<entry colname="col2">The number of rows in the base table does not match
the number of rows in at least 1 of the indexes on the table. Index '&lt;<i>value</i>>'
on table '&lt;<i>value</i>>.&lt;<i>value</i>>' has &lt;<i>value</i>> rows,
but the base table has &lt;<i>value</i>> rows.  The suggested corrective action
is to recreate the index.  </entry>
</row>
<row>
<entry colname="col1">X0Y56</entry>
<entry colname="col2">'&lt;<i>value</i>>' is not allowed on the System table
'&lt;<i>value</i>>'.</entry>
</row>
<row>
<entry colname="col1">X0Y57</entry>
<entry colname="col2">A non-nullable column cannot be added to table '&lt;<i>tableName</i>>'
because the table contains at least one row.  Non-nullable columns can only
be added to empty tables.   </entry>
</row>
<row>
<entry colname="col1">X0Y58</entry>
<entry colname="col2">Attempt to add a primary key constraint to table '&lt;<i>tableName</i>>'
failed because the table already has a constraint of that type.  A table can
only have a single primary key constraint.   </entry>
</row>
<row>
<entry colname="col1">X0Y59       </entry>
<entry colname="col2">Attempt to add or enable constraint(s) on table '&lt;<i>tableName</i>>'
failed because the table contains &lt;<i>rowName</i>> row(s) that violate
the following check constraint(s): &lt;<i>constraintName</i>>. </entry>
</row>
<row>
<entry colname="col1">X0Y63</entry>
<entry colname="col2">The command on table '&lt;<i>tableName</i>>' failed
because null data was found in the primary key or unique constraint/index
column(s). All columns in a primary or unique index key must not be null.
 </entry>
</row>
<row>
<entry colname="col1">X0Y66     </entry>
<entry colname="col2">Cannot issue commit in a nested connection when there
is a pending operation in the parent connection. </entry>
</row>
<row>
<entry colname="col1">X0Y67     </entry>
<entry colname="col2">Cannot issue rollback in a nested connection when there
is a pending operation in the parent connection.          </entry>
</row>
<row>
<entry colname="col1">X0Y68          </entry>
<entry colname="col2">&lt;<i>value</i>> '&lt;<i>value</i>>' already exists.
 </entry>
</row>
<row>
<entry colname="col1">X0Y69</entry>
<entry colname="col2">&lt;<i>value</i>> is not permitted because trigger &lt;<i>triggerName</i>>
is active on &lt;<i>value</i>>.      </entry>
</row>
<row>
<entry colname="col1">X0Y70   </entry>
<entry colname="col2">INSERT, UPDATE and DELETE are not permitted on table &lt;<i>tableName</i>>
because trigger &lt;<i>triggerName</i>> is active. </entry>
</row>
<row>
<entry colname="col1">X0Y71</entry>
<entry colname="col2">Transaction manipulation such as SET ISOLATION is not
permitted because trigger &lt;<i>triggerName</i>> is active.              </entry>
</row>
<row>
<entry colname="col1">X0Y72</entry>
<entry colname="col2">Bulk insert replace is not permitted on '&lt;<i>value</i>>'
because it has an enabled trigger (&lt;<i>value</i>>).   </entry>
</row>
<row>
<entry colname="col1">X0Y77</entry>
<entry colname="col2">Cannot issue set transaction isolation statement on
a global transaction that is in progress because it would have implicitly
committed the global transaction.  </entry>
</row>
<row>
<entry colname="col1">X0Y78</entry>
<entry colname="col2">Statement.executeQuery() cannot be called with a statement
that returns a row count.  </entry>
</row>
<row>
<entry colname="col1">X0Y79</entry>
<entry colname="col2">Statement.executeUpdate() cannot be called with a statement
that returns a ResultSet.   </entry>
</row>
<row>
<entry colname="col1">X0Y80 </entry>
<entry colname="col2">ALTER table '&lt;<i>tableName</i>>' failed. Null data
found in column '&lt;<i>columnName</i>>'.    </entry>
</row>
<row>
<entry colname="col1">X0Y83
                           </entry>
<entry colname="col2">WARNING: While deleting a row from a table the index
row for base table row &lt;<i>rowName</i>> was not found in index with conglomerate
id &lt;<i>id</i>>.  This problem has automatically been corrected as part
of the delete operation.     </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XBCA: CacheService</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row valign="bottom">
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XBCA0</entry>
<entry colname="col2">Cannot create new object with key &lt;<i>key</i>> in &lt;<i>cache</i>>
cache. The object already exists in the cache.   </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XBCM: ClassManager</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">XBCM1   </entry>
<entry colname="col2">Java linkage error thrown during load of generated class &lt;<i>className</i>>. </entry>
</row>
<row>
<entry colname="col1">XBCM2</entry>
<entry colname="col2">Cannot create an instance of generated class &lt;<i>className</i>>. </entry>
</row>
<row>
<entry colname="col1">XBCM3</entry>
<entry colname="col2">Method &lt;<i>methodName</i>> () does not exist in generated
class &lt;<i>className</i>>. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XBCX: Criptografia</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">XBCX0</entry>
<entry colname="col2">Exception from Cryptography provider. See next exception
for details.</entry>
</row>
<row>
<entry colname="col1">XBCX1</entry>
<entry colname="col2">Initializing cipher with illegal mode, must be either
ENCRYPT or DECRYPT.     </entry>
</row>
<row>
<entry colname="col1">XBCX2 </entry>
<entry colname="col2">Initializing cipher with a boot password that is too
short. The password must be at least &lt;<i>number</i>> characters long.
  </entry>
</row>
<row>
<entry colname="col1">XBCX5</entry>
<entry colname="col2">Cannot change boot password to null.   </entry>
</row>
<row>
<entry colname="col1">XBCX6     </entry>
<entry colname="col2">Cannot change boot password to a non-string serializable
type.         </entry>
</row>
<row>
<entry colname="col1">XBCX7      </entry>
<entry colname="col2">Wrong format for changing boot password.  Format must
be : <codeph>old_boot_password, new_boot_password</codeph>.          </entry>
</row>
<row>
<entry colname="col1">XBCX8 </entry>
<entry colname="col2">Cannot change boot password for a non-encrypted database.
       </entry>
</row>
<row>
<entry colname="col1">XBCX9</entry>
<entry colname="col2">Cannot change boot password for a read-only database.
     </entry>
</row>
<row>
<entry colname="col1">XBCXA</entry>
<entry colname="col2">Wrong boot password.        </entry>
</row>
<row>
<entry colname="col1">XBCXB</entry>
<entry colname="col2">Bad encryption padding '&lt;<i>value</i>>' or padding
not specified. 'NoPadding' must be used.    </entry>
</row>
<row>
<entry colname="col1">XBCXC</entry>
<entry colname="col2">Encryption algorithm '&lt;<i>algorithmName</i>>' does
not exist. Please check that the chosen provider '&lt;<i>providerName</i>>'
supports this algorithm.</entry>
</row>
<row>
<entry colname="col1">XBCXD</entry>
<entry colname="col2">The encryption algorithm cannot be changed after the
database is created. </entry>
</row>
<row>
<entry colname="col1">XBCXE    </entry>
<entry colname="col2">The encryption provider cannot be changed after the
database is created.   </entry>
</row>
<row>
<entry colname="col1">XBCXF    </entry>
<entry colname="col2">The class '&lt;<i>className</i>>' representing the encryption
provider cannot be found. </entry>
</row>
<row>
<entry colname="col1">XBCXG</entry>
<entry colname="col2">The encryption provider '&lt;<i>providerName</i>>' does
not exist.    </entry>
</row>
<row>
<entry colname="col1">XBCXH</entry>
<entry colname="col2">The encryptionAlgorithm '&lt;<i>algorithmName</i>>'
is not in the correct format. The correct format is <codeph>algorithm/feedbackMode/NoPadding</codeph>.
            </entry>
</row>
<row>
<entry colname="col1">XBCXI</entry>
<entry colname="col2">The feedback mode '&lt;<i>mode</i>>' is not supported.
Supported feedback modes are CBC, CFB, OFB and ECB.  </entry>
</row>
<row>
<entry colname="col1">XBCXJ</entry>
<entry colname="col2">The application is using a version of the Java Cryptography
Extension (JCE) earlier than 1.2.1.  Please upgrade to JCE 1.2.1 and try the
operation again.    </entry>
</row>
<row>
<entry colname="col1">XBCXK</entry>
<entry colname="col2">The given encryption key does not match the encryption
key used when creating the database. Please ensure that you are using the
correct encryption key and try again.      </entry>
</row>
<row>
<entry colname="col1">XBCXL</entry>
<entry colname="col2">The verification process for the encryption key was
not successful. This could have been caused by an error when accessing the
appropriate file to do the verification process.  See next exception for details.
 </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XBM: Monitor</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">XBM01   </entry>
<entry colname="col2">Startup failed due to an exception. See next exception
for details. </entry>
</row>
<row>
<entry colname="col1">XBM02</entry>
<entry colname="col2">Startup failed due to missing functionality for &lt;<i>value</i>>.
Please ensure your classpath includes the correct <ph conref="refconrefs.dita#prod/productshortname"></ph> Derby
software.            </entry>
</row>
<row>
<entry colname="col1">XBM05</entry>
<entry colname="col2">Startup failed due to missing product version information
for &lt;<i>value</i>>.      </entry>
</row>
<row>
<entry colname="col1">XBM06</entry>
<entry colname="col2">Startup failed. An encrypted database cannot be accessed
without the correct boot password.  </entry>
</row>
<row>
<entry colname="col1">XBM07</entry>
<entry colname="col2">Startup failed. Boot password must be at least 8 bytes
long.</entry>
</row>
<row>
<entry colname="col1">XBM08</entry>
<entry colname="col2">Could not instantiate &lt;<i>value</i>> StorageFactory
class &lt;<i>value</i>>.   </entry>
</row>
<row>
<entry colname="col1">XBM0G</entry>
<entry colname="col2">Failed to start encryption engine. Please make sure
you are running Java 2 and have downloaded an encryption provider such as
jce and put it in your classpath.         </entry>
</row>
<row>
<entry colname="col1">XBM0H</entry>
<entry colname="col2">Directory &lt;<i>directoryName</i>> cannot be created.
     </entry>
</row>
<row>
<entry colname="col1">XBM0I</entry>
<entry colname="col2">Directory &lt;<i>directoryName</i>> cannot be removed.
   </entry>
</row>
<row>
<entry colname="col1">XBM0J</entry>
<entry colname="col2">Directory &lt;<i>directoryName</i>> already exists.
          </entry>
</row>
<row>
<entry colname="col1">XBM0K</entry>
<entry colname="col2">Unknown sub-protocol for database name &lt;<i>databaseName</i>>.
 </entry>
</row>
<row>
<entry colname="col1">XBM0L</entry>
<entry colname="col2">Specified authentication scheme class &lt;<i>className</i>>
does implement the authentication interface &lt;<i>interfaceName</i>>.   </entry>
</row>
<row>
<entry colname="col1">XBM0M</entry>
<entry colname="col2">Error creating instance of authentication scheme class &lt;<i>className</i>>.
   </entry>
</row>
<row>
<entry colname="col1">XBM0N</entry>
<entry colname="col2">JDBC Driver registration with java.sql.DriverManager
failed. See next exception for details.   </entry>
</row>
<row>
<entry colname="col1">XBM0P</entry>
<entry colname="col2">Service provider is read-only. Operation not permitted.
    </entry>
</row>
<row>
<entry colname="col1">XBM0Q</entry>
<entry colname="col2">File &lt;<i>fileName</i>> not found. Please make sure
that backup copy is the correct one and it is not corrupted.
   </entry>
</row>
<row>
<entry colname="col1">XBM0R</entry>
<entry colname="col2">Unable to remove file &lt;<i>fileName</i>>.      </entry>
</row>
<row>
<entry colname="col1">XBM0S</entry>
<entry colname="col2">Unable to rename file '&lt;<i>fileName</i>>' to '<i>fileName</i>>'
      </entry>
</row>
<row>
<entry colname="col1">XBM0T</entry>
<entry colname="col2">Ambiguous sub-protocol for database name &lt;<i>databaseName</i>>.</entry>
</row>
<row>
<entry colname="col1">XBM0U  </entry>
<entry colname="col2">No class was registered for identifier &lt;<i>identifierName</i>>.</entry>
</row>
<row>
<entry colname="col1">XBM0V  </entry>
<entry colname="col2">An exception was thrown while loading class &lt;<i>className</i>>
registered for identifier &lt;<i>identifierName</i>>.     </entry>
</row>
<row>
<entry colname="col1">XBM0W</entry>
<entry colname="col2">An exception was thrown while creating an instance of
class &lt;<i>className</i>> registered for identifier &lt;<i>identifierName</i>>.
   </entry>
</row>
<row>
<entry colname="col1">XBM0X</entry>
<entry colname="col2">Supplied territory description '&lt;<i>value</i>>' is
invalid, expecting ln[_CO[_variant]] ln=lower-case two-letter ISO-639 language
code, CO=upper-case two-letter ISO-3166 country codes, see java.util.Locale.
      </entry>
</row>
<row>
<entry colname="col1">XBM0Y</entry>
<entry colname="col2">Backup database directory &lt;<i>directoryName</i>>
not found. Please make sure that the specified backup path is right.
  </entry>
</row>
<row>
<entry colname="col1">XBM0Z</entry>
<entry colname="col2">Unable to copy file '&lt;<i>value</i>>' to '&lt;<i>value</i>>'.
Please make sure that there is enough space and permissions are correct.
  </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XCL: Non-SQLSTATE</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">XCL01</entry>
<entry colname="col2">ResultSet does not return rows. Operation &lt;<i>operationName</i>>
not permitted. </entry>
</row>
<row>
<entry colname="col1">XCL05</entry>
<entry colname="col2">Activation closed. Operation &lt;<i>operationName</i>>
not permitted.  </entry>
</row>
<row>
<entry colname="col1">XCL07</entry>
<entry colname="col2">Cursor '&lt;<i>cursorName</i>>' is closed. Verify that
autocommit is OFF.    </entry>
</row>
<row>
<entry colname="col1">XCL08</entry>
<entry colname="col2">Cursor '&lt;<i>cursorName</i>>' is not on a row.  </entry>
</row>
<row>
<entry colname="col1">XCL09   </entry>
<entry colname="col2">An Activation was passed to the '&lt;<i>methodName</i>>'
method that does not match the PreparedStatement. </entry>
</row>
<row>
<entry colname="col1">XCL10</entry>
<entry colname="col2">A PreparedStatement has been recompiled and the parameters
have changed.  If you are using JDBC you must prepare the statement again. </entry>
</row>
<row>
<entry colname="col1">XCL12</entry>
<entry colname="col2">An attempt was made to put a data value of type '&lt;<i>type</i>>'
into a data value of type '&lt;<i>type</i>>'.  </entry>
</row>
<row>
<entry colname="col1">XCL13</entry>
<entry colname="col2">The parameter position '&lt;<i>parameterPosition</i>>'
is out of range.  The number of parameters for this prepared  statement is
'&lt;<i>number</i>>'.    </entry>
</row>
<row>
<entry colname="col1">XCL14</entry>
<entry colname="col2">The column position '&lt;<i>value</i>>' is out of range.
 The number of columns for this ResultSet is '&lt;<i>number</i>>'.</entry>
</row>
<row>
<entry colname="col1">XCL15</entry>
<entry colname="col2">A ClassCastException occurred when calling the compareTo()
method on an object '&lt;<i>object</i>>'.  The parameter to compareTo() is
of class '<i>className</i>>'. </entry>
</row>
<row>
<entry colname="col1">XCL16</entry>
<entry colname="col2">ResultSet not open. Operation '&lt;<i>operation</i>>'
not permitted. Verify that autocommit is OFF.   </entry>
</row>
<row>
<entry colname="col1">XCL17</entry>
<entry colname="col2">Statement not allowed in this database.   </entry>
</row>
<row>
<entry colname="col1">XCL19</entry>
<entry colname="col2">Missing row in table '&lt;<i>tableName</i>>' for key
'&lt;<i>key</i>>'. </entry>
</row>
<row>
<entry colname="col1">XCL20</entry>
<entry colname="col2">Catalogs at version level '&lt;<i>versionLevel</i>>'
cannot be upgraded to version level '&lt;<i>versionLevel</i>>'.</entry>
</row>
<row>
<entry colname="col1">XCL21</entry>
<entry colname="col2">You are trying to execute a Data Definition statement
(CREATE, DROP, or ALTER) while preparing a different statement. This is not
allowed. It can happen if you execute a Data Definition statement from within
a static initializer of a Java class that is being used from within a SQL
statement.</entry>
</row>
<row>
<entry colname="col1">XCL22</entry>
<entry colname="col2">Parameter &lt;<i>parameterName</i>> cannot be registered
as an OUT parameter because it is an IN parameter. </entry>
</row>
<row>
<entry colname="col1">XCL23</entry>
<entry colname="col2">SQL type number '&lt;<i>type</i>>' is not a supported
type by registerOutParameter().   </entry>
</row>
<row>
<entry colname="col1">XCL24</entry>
<entry colname="col2">Parameter &lt;<i>parameterName</i>> appears to be an
output parameter, but it has not been so designated by registerOutParameter().
 If it is not an output parameter, then it has to be set to type &lt;<i>type</i>>.
  </entry>
</row>
<row>
<entry colname="col1">XCL25</entry>
<entry colname="col2">Parameter &lt;<i>parameterName</i>> cannot be registered
to be of type &lt;<i>type</i>> because it maps to type &lt;<i>type</i>> and
they are incompatible. </entry>
</row>
<row>
<entry colname="col1">XCL26 </entry>
<entry colname="col2">Parameter &lt;<i>parameterName</i>> is not an output
parameter. </entry>
</row>
<row>
<entry colname="col1">XCL27</entry>
<entry colname="col2">Return output parameters cannot be set.   </entry>
</row>
<row>
<entry colname="col1">XCL30</entry>
<entry colname="col2">An IOException was thrown when reading a '&lt;<i>value</i>>'
from an InputStream. </entry>
</row>
<row>
<entry colname="col1">XCL31</entry>
<entry colname="col2">Statement closed. </entry>
</row>
<row>
<entry colname="col1">XCL33</entry>
<entry colname="col2">The table cannot be defined as a dependent of table &lt;<i>tableName</i>>
because of delete rule restrictions. (The relationship is self-referencing
and a self-referencing relationship already exists with the SET NULL delete
rule.)     </entry>
</row>
<row>
<entry colname="col1">XCL34</entry>
<entry colname="col2">The table cannot be defined as a dependent of table &lt;<i>tableName</i>>
because of delete rule restrictions. (The relationship forms a cycle of two
or more tables that cause the table to be delete-connected to itself (all
other delete rules in the cycle would be CASCADE)).    </entry>
</row>
<row>
<entry colname="col1">XCL35</entry>
<entry colname="col2">The table cannot be defined as a dependent of table &lt;<i>tableName</i>>
because of delete rule restrictions. (The relationship causes the table to
be delete-connected to the indicated table through multiple relationships
and the delete rule of the existing relationship is SET NULL).   </entry>
</row>
<row>
<entry colname="col1">XCL36</entry>
<entry colname="col2">The delete rule of foreign key must be &lt;<i>ruleName</i>>.
(The referential constraint is self-referencing and an existing self-referencing
constraint has the indicated delete rule (NO ACTION, RESTRICT or CASCADE).)
  </entry>
</row>
<row>
<entry colname="col1">XCL37</entry>
<entry colname="col2">The delete rule of foreign key must be &lt;<i>ruleName</i>>.
(The referential constraint is self-referencing and the table is dependent
in a relationship with a delete rule of CASCADE.)  </entry>
</row>
<row>
<entry colname="col1">XCL38</entry>
<entry colname="col2">The delete rule of foreign key  must be &lt;<i>ruleName</i>>.
(The relationship would cause the table to be delete-connected to the same
table through multiple relationships and such relationships must have the
same delete rule (NO ACTION, RESTRICT or CASCADE).)     </entry>
</row>
<row>
<entry colname="col1">XCL39</entry>
<entry colname="col2">The delete rule of foreign key cannot be CASCADE. (A
self-referencing constraint exists with a delete rule of SET NULL, NO ACTION
or RESTRICT.)    </entry>
</row>
<row>
<entry colname="col1">XCL40</entry>
<entry colname="col2">The delete rule of foreign key cannot be CASCADE. (The
relationship would form a cycle that would cause a table to be delete-connected
to itself. One of the existing delete rules in the cycle is not CASCADE, so
this relationship may be definable if the delete rule is not CASCADE.)  </entry>
</row>
<row>
<entry colname="col1">XCL41</entry>
<entry colname="col2">the delete rule of foreign key can not be CASCADE. (The
relationship would cause another table to be delete-connected to the same
table through multiple paths with different delete rules or with delete rule
equal to SET NULL.)   </entry>
</row>
<row>
<entry colname="col1">XCL42</entry>
<entry colname="col2">CASCADE</entry>
</row>
<row>
<entry colname="col1">XCL43</entry>
<entry colname="col2">SET NULL</entry>
</row>
<row>
<entry colname="col1">XCL44</entry>
<entry colname="col2">RESTRICT   </entry>
</row>
<row>
<entry colname="col1">XCL45</entry>
<entry colname="col2">NO ACTION   </entry>
</row>
<row>
<entry colname="col1">XCL46</entry>
<entry colname="col2">SET DEFAULT</entry>
</row>
<row>
<entry colname="col1">XCL47</entry>
<entry colname="col2">Use of '&lt;<i>value</i>>' requires database to be upgraded
from version &lt;<i>versionNumber</i>> to version &lt;<i>versionNumber</i>>
or later.   </entry>
</row>
<row>
<entry colname="col1">XCL48</entry>
<entry colname="col2">TRUNCATE TABLE is not permitted on '&lt;<i>value</i>>'
because unique/primary key constraints on this table are referenced by enabled
foreign key constraints from other tables.  </entry>
</row>
<row>
<entry colname="col1">XCL49</entry>
<entry colname="col2">TRUNCATE TABLE is not permitted on '&lt;<i>value</i>>'
because it has an enabled DELETE trigger (&lt;<i>value</i>>).   </entry>
</row>
<row>
<entry colname="col1">XCL50</entry>
<entry colname="col2">Upgrading the database from a previous version is not
supported.  The database being accessed is at version level '<i>versionNumber</i>>',
this software is at version level '&lt;<i>versionNumber</i>>'.    </entry>
</row>
<row>
<entry colname="col1">XCL51</entry>
<entry colname="col2">The requested function cannot reference tables in SESSION
schema.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XCW: Atualização não suportada</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">XCW00</entry>
<entry colname="col2">Unsupported upgrade from '&lt;<i>value</i>>' to '&lt;<i>value</i>>'. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XCXA: Erro de análise de ID</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XCXA0</entry>
<entry colname="col2">Invalid identifier.  </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XCXB: DB_Class_Path_Parse_Error</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XCXB0</entry>
<entry colname="col2">Invalid database classpath: '&lt;<i>classpath</i>>'.
 </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XCXC: Erro de análise de lista de ID</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">XCXC0</entry>
<entry colname="col2">Invalid id list.   </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XCXE: Sem idioma</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XCXE0</entry>
<entry colname="col2">You are trying to do an operation that uses the territory
of the database, but the database does not have a territory.  </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XCY: Propriedades</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">XCY00</entry>
<entry colname="col2">Invalid value for property '&lt;<i>value</i>>'='&lt;<i>value</i>>'. </entry>
</row>
<row>
<entry colname="col1">XCY02</entry>
<entry colname="col2">The requested property change is not supported '&lt;<i>value</i>>'='&lt;<i>value</i>>'.
 </entry>
</row>
<row>
<entry colname="col1">XCY03</entry>
<entry colname="col2">Required property '&lt;<i>propertyName</i>>' has not
been set. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XCZ: org.apache.derby.database.UserUtility</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">XCZ00</entry>
<entry colname="col2">Unknown permission '&lt;<i>permissionName</i>>'.  </entry>
</row>
<row>
<entry colname="col1">XCZ01</entry>
<entry colname="col2">Unknown user '&lt;<i>userName</i>>'.   </entry>
</row>
<row>
<entry colname="col1">XCZ02</entry>
<entry colname="col2">Invalid parameter '&lt;<i>value</i>>'='&lt;<i>value</i>>'. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XD00x: Gerenciador de dependências</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">XD003</entry>
<entry colname="col2">Unable to restore dependency from disk. DependableFinder
= '&lt;<i>value</i>>'. Further information: '&lt;<i>value</i>>'. </entry>
</row>
<row>
<entry colname="col1">XD004</entry>
<entry colname="col2">Unable to store dependencies. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XIE: Importação/Exportação</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">XIE01</entry>
<entry colname="col2">Connection was null.  </entry>
</row>
<row>
<entry colname="col1">XIE03</entry>
<entry colname="col2">Data found on line &lt;<i>lineNumber</i>> for column &lt;<i>columnName</i>>
after the stop delimiter.  </entry>
</row>
<row>
<entry colname="col1">XIE04</entry>
<entry colname="col2">Data file not found: &lt;<i>fileName</i>>. </entry>
</row>
<row>
<entry colname="col1">XIE05</entry>
<entry colname="col2">Data file cannot be null. </entry>
</row>
<row>
<entry colname="col1">XIE06</entry>
<entry colname="col2">Entity name was null.  </entry>
</row>
<row>
<entry colname="col1">XIE07</entry>
<entry colname="col2">Field and record separators cannot be substrings of
each other. </entry>
</row>
<row>
<entry colname="col1">XIE08</entry>
<entry colname="col2">There is no column named: &lt;<i>columnName</i>>.</entry>
</row>
<row>
<entry colname="col1">XIE09</entry>
<entry colname="col2">The total number of columns in the row is: &lt;<i>number</i>>.
 </entry>
</row>
<row>
<entry colname="col1">XIE0B</entry>
<entry colname="col2">Column '&lt;<i>columnName</i>>' in the table is of type &lt;<i>type</i>>,
it is not supported by the import/export feature.   </entry>
</row>
<row>
<entry colname="col1">XIE0D</entry>
<entry colname="col2">Cannot find the record separator on line &lt;<i>lineNumber</i>>.</entry>
</row>
<row>
<entry colname="col1">XIE0E</entry>
<entry colname="col2">Read endOfFile at unexpected place on line &lt;<i>lineNumber</i>.
   </entry>
</row>
<row>
<entry colname="col1">XIE0I                                             </entry>
<entry colname="col2">An IOException occurred while writing data to the file.</entry>
</row>
<row>
<entry colname="col1">XIE0J</entry>
<entry colname="col2">A delimiter is not valid or is used more than once. </entry>
</row>
<row>
<entry colname="col1">XIE0K</entry>
<entry colname="col2">The period was specified as a character string delimiter.
 </entry>
</row>
<row>
<entry colname="col1">XIE0M</entry>
<entry colname="col2">Table '&lt;<i>tableName</i>>' does not exist.       </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XJ: Erros de conectividade</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">XJ004</entry>
<entry colname="col2">Database '&lt;<i>databaseName</i>>' not found.  </entry>
</row>
<row>
<entry colname="col1">XJ009 </entry>
<entry colname="col2">Use of CallableStatement required for stored procedure
call or use of output parameters: &lt;<i>value</i>>   </entry>
</row>
<row>
<entry colname="col1">XJ010    </entry>
<entry colname="col2">Cannot issue savepoint when autoCommit is on.       </entry>
</row>
<row>
<entry colname="col1">XJ011</entry>
<entry colname="col2">Cannot pass null for savepoint name. </entry>
</row>
<row>
<entry colname="col1">XJ012</entry>
<entry colname="col2">'&lt;<i>value</i>>' already closed.   </entry>
</row>
<row>
<entry colname="col1">XJ013 </entry>
<entry colname="col2">No ID for named savepoints.    </entry>
</row>
<row>
<entry colname="col1">XJ014</entry>
<entry colname="col2">No name for un-named savepoints.   </entry>
</row>
<row>
<entry colname="col1">XJ015</entry>
<entry colname="col2">Derby system shutdown.  </entry>
</row>
<row>
<entry colname="col1">XJ016</entry>
<entry colname="col2">Method '&lt;<i>methodName</i>>' not allowed on prepared
statement.  </entry>
</row>
<row>
<entry colname="col1">XJ017</entry>
<entry colname="col2">No savepoint command allowed inside the trigger code.
  </entry>
</row>
<row>
<entry colname="col1">XJ018</entry>
<entry colname="col2">Column name cannot be null. </entry>
</row>
<row>
<entry colname="col1">XJ020     </entry>
<entry colname="col2">Object type not convertible to TYPE '&lt;<i>typeName</i>>',
invalid java.sql.Types value, or object was null. </entry>
</row>
<row>
<entry colname="col1">XJ022    </entry>
<entry colname="col2">Unable to set stream: '&lt;<i>name</i>>'.         </entry>
</row>
<row>
<entry colname="col1">XJ023</entry>
<entry colname="col2">Input stream held less data than requested length.  </entry>
</row>
<row>
<entry colname="col1">XJ025</entry>
<entry colname="col2">Input stream cannot have negative length.  </entry>
</row>
<row>
<entry colname="col1">XJ028 </entry>
<entry colname="col2">The URL '&lt;<i>urlValue</i>>' is not properly formed.</entry>
</row>
<row>
<entry colname="col1">XJ030</entry>
<entry colname="col2">Cannot set AUTOCOMMIT ON when in a nested connection.
  </entry>
</row>
<row>
<entry colname="col1">XJ040</entry>
<entry colname="col2">Failed to start database '&lt;<i>databaseName</i>>',
see the next exception for details.   </entry>
</row>
<row>
<entry colname="col1">XJ041</entry>
<entry colname="col2">Failed to create database '&lt;<i>databaseName</i>>',
see the next exception for details. </entry>
</row>
<row>
<entry colname="col1">XJ042    </entry>
<entry colname="col2">'&lt;<i>value</i>>' is not a valid value for property
'&lt;<i>propertyName</i>>'.</entry>
</row>
<row>
<entry colname="col1">XJ044</entry>
<entry colname="col2">'&lt;<i>value</i>>' is an invalid scale.</entry>
</row>
<row>
<entry colname="col1">XJ045</entry>
<entry colname="col2">Invalid or (currently) unsupported isolation level,
'&lt;<i>value</i>>', passed to Connection.setTransactionIsolationLevel().
The currently supported values are java.sql.Connection.TRANSACTION_SERIALIZABLE,
java.sql.Connection.TRANSACTION_REPEATABLE_READ, java.sql.Connection.TRANSACTION_READ_COMMITTED,
and java.sql.Connection.TRANSACTION_READ_UNCOMMITTED.     </entry>
</row>
<row>
<entry colname="col1">XJ049</entry>
<entry colname="col2">Conflicting create attributes specified.</entry>
</row>
<row>
<entry colname="col1">XJ04B</entry>
<entry colname="col2">Batch cannot contain a command that attempts to return
a result set.</entry>
</row>
<row>
<entry colname="col1">XJ04C</entry>
<entry colname="col2">CallableStatement batch cannot contain output parameters.</entry>
</row>
<row>
<entry colname="col1">XJ056</entry>
<entry colname="col2">Cannot set AUTOCOMMIT ON when in an XA connection.</entry>
</row>
<row>
<entry colname="col1">XJ057</entry>
<entry colname="col2">Cannot commit a global transaction using the Connection.
Commit processing must go through XAResource interface. </entry>
</row>
<row>
<entry colname="col1">XJ058</entry>
<entry colname="col2">Cannot rollback a global transaction using the Connection,
commit processing must go through XAResource interface.     </entry>
</row>
<row>
<entry colname="col1">XJ059</entry>
<entry colname="col2">Cannot close a connection while a global transaction
is still active.     </entry>
</row>
<row>
<entry colname="col1">XJ05B</entry>
<entry colname="col2">JDBC attribute '&lt;<i>attributeName</i>>' has an invalid
value '&lt;<i>value</i>>', Valid values are '&lt;<i>value</i>>'.</entry>
</row>
<row>
<entry colname="col1">XJ05C </entry>
<entry colname="col2">Cannot set holdability ResultSet.HOLD_CURSORS_OVER_COMMIT
for a global transaction.   </entry>
</row>
<row>
<entry colname="col1">XJ061</entry>
<entry colname="col2">The '&lt;<i>valueName</i>>' method is only allowed on
scroll cursors.   </entry>
</row>
<row>
<entry colname="col1">XJ062</entry>
<entry colname="col2">Invalid parameter value '&lt;<i>value</i>>' for ResultSet.setFetchSize(int
rows). </entry>
</row>
<row>
<entry colname="col1">XJ063</entry>
<entry colname="col2">Invalid parameter value '&lt;<i>value</i>>' for Statement.setMaxRows(int
maxRows).  Parameter value must be >= 0.</entry>
</row>
<row>
<entry colname="col1">XJ064</entry>
<entry colname="col2">Invalid parameter value '&lt;<i>value</i>>' for setFetchDirection(int
direction). </entry>
</row>
<row>
<entry colname="col1">XJ065</entry>
<entry colname="col2">Invalid parameter value '&lt;<i>value</i>>' for Statement.setFetchSize(int
rows).  </entry>
</row>
<row>
<entry colname="col1">XJ066</entry>
<entry colname="col2">Invalid parameter value '&lt;<i>value</i>>' for Statement.setMaxFieldSize(int
max).</entry>
</row>
<row>
<entry colname="col1">XJ067</entry>
<entry colname="col2">SQL text pointer is null.        </entry>
</row>
<row>
<entry colname="col1">XJ068</entry>
<entry colname="col2">Only executeBatch and clearBatch allowed in the middle
of a batch.</entry>
</row>
<row>
<entry colname="col1">XJ069</entry>
<entry colname="col2">No SetXXX methods allowed in case of USING execute statement.</entry>
</row>
<row>
<entry colname="col1">XJ070</entry>
<entry colname="col2">Negative or zero position argument '&lt;<i>argument</i>>'
passed in a Blob or Clob method.</entry>
</row>
<row>
<entry colname="col1">XJ071</entry>
<entry colname="col2">Zero or negative length argument '&lt;<i>argument</i>>'
passed in a BLOB or CLOB method.</entry>
</row>
<row>
<entry colname="col1">XJ072</entry>
<entry colname="col2">Null pattern or searchStr passed in to a BLOB or CLOB
position method.</entry>
</row>
<row>
<entry colname="col1">XJ073</entry>
<entry colname="col2">The data in this BLOB or CLOB is no longer available.
The BLOB or CLOB's transaction may be committed, or its connection is closed.</entry>
</row>
<row>
<entry colname="col1">XJ076</entry>
<entry colname="col2">The position argument '&lt;<i>argument</i>>' exceeds
the size of the BLOB/CLOB.</entry>
</row>
<row>
<entry colname="col1">XJ077</entry>
<entry colname="col2">Got an exception when trying to read the first byte/character
of the BLOB/CLOB pattern using getBytes/getSubString.</entry>
</row>
<row>
<entry colname="col1">XJ080</entry>
<entry colname="col2">USING execute statement passed &lt;<i>numparameters</i>>
parameters rather than &lt;<i>numparameters</i>>.</entry>
</row>
<row>
<entry colname="col1">XJ081</entry>
<entry colname="col2">Conflicting create/restore/recovery attributes specified.</entry>
</row>
<row>
<entry colname="col1">XJ081</entry>
<entry colname="col2">Invalid value '&lt;<i>value</i>>' passed as parameter
'&lt;<i>parameterName</i>>' to method '&lt;<i>moduleName</i>>'.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSAIx: Armazenamento - access.protocol.interface
statement exceptions</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row valign="bottom">
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSAI2</entry>
<entry colname="col2">The conglomerate (&lt;<i>value</i>>) requested does
not exist.</entry>
</row>
<row>
<entry colname="col1">XSAI3</entry>
<entry colname="col2">Feature not implemented.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSAMx: Armazenamento - AccessManager</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1"> XSAM0 </entry>
<entry colname="col2">Exception encountered while trying to boot module for
'&lt;<i>value</i>>'.</entry>
</row>
<row>
<entry colname="col1">XSAM2 </entry>
<entry colname="col2">There is no index or conglomerate with conglom id '&lt;<i>conglomID</i>>'
to drop. </entry>
</row>
<row>
<entry colname="col1">XSAM3</entry>
<entry colname="col2">There is no index or conglomerate with conglom id '&lt;<i>conglomID</i>>'.</entry>
</row>
<row>
<entry colname="col1">XSAM4 </entry>
<entry colname="col2">There is no sort called '&lt;<i>sortName</i>>'.</entry>
</row>
<row>
<entry colname="col1">XSAM5 </entry>
<entry colname="col2">Scan must be opened and positioned by calling next()
before making other calls.</entry>
</row>
<row>
<entry colname="col1">XSAM6 </entry>
<entry colname="col2">Record &lt;<i>recordnumber</i>> on page &lt;<i>page</i>>
in container &lt;<i>containerName</i>> not found.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSASx: Armazenamento - Sort</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSAS0 </entry>
<entry colname="col2">A scan controller interface method was called which
is not appropriate for a scan on a sort.</entry>
</row>
<row valign="top">
<entry colname="col1"> XSAS1 </entry>
<entry colname="col2">An attempt was made to fetch a row before the beginning
of a sort or after the end of a sort.  </entry>
</row>
<row>
<entry colname="col1">XSAS3 </entry>
<entry colname="col2">The type of a row inserted into a sort does not match
the sorts template. </entry>
</row>
<row>
<entry colname="col1">XSAS6 </entry>
<entry colname="col2">Could not acquire resources for sort. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSAXx: Armazenamento - access.protocol.XA statement
exception</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSAX0 </entry>
<entry colname="col2">XA protocol violation.</entry>
</row>
<row valign="top">
<entry colname="col1">XSAX1 </entry>
<entry colname="col2">An attempt was made to start a global transaction with
an Xid of an existing global transaction. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSCBx: Armazenamento - BTree</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row valign="top">
<entry colname="col1">XSCB0</entry>
<entry colname="col2">Could not create container. </entry>
</row>
<row>
<entry colname="col1">XSCB1 </entry>
<entry colname="col2">Container &lt;<i>containerName</i>> not found.   </entry>
</row>
<row>
<entry colname="col1">XSCB2 </entry>
<entry colname="col2">The required property &lt;<i>propertyName</i>> not found
in the property list given to createConglomerate() for a btree secondary index. </entry>
</row>
<row>
<entry colname="col1">XSCB3 </entry>
<entry colname="col2">Unimplemented feature.</entry>
</row>
<row>
<entry colname="col1">XSCB4 </entry>
<entry colname="col2">A method on a btree open scan was called prior to positioning
the scan on the first row (that is, no next() call has been made yet).  The
current state of the scan is (&lt;<i>value</i>>).</entry>
</row>
<row>
<entry colname="col1">XSCB5 </entry>
<entry colname="col2">During logical undo of a btree insert or delete, the
row could not be found in the tree. </entry>
</row>
<row>
<entry colname="col1">XSCB6 </entry>
<entry colname="col2">Limitation: Record of a btree secondary index cannot
be updated or inserted due to lack of space on the page. Use the parameters
derby.storage.pageSize and/or derby.storage.pageReservedSpace to work around
this limitation. </entry>
</row>
<row>
<entry colname="col1">XSCB7 </entry>
<entry colname="col2">An internal error was encountered during a btree scan
- current_rh is null = &lt;<i>value</i>>, position key is null = &lt;<i>value</i>>. </entry>
</row>
<row>
<entry colname="col1">XSCB8 </entry>
<entry colname="col2">The btree conglomerate &lt;<i>value</i>> is closed. </entry>
</row>
<row>
<entry colname="col1">XSCB9 </entry>
<entry colname="col2">Reserved for testing.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSCG0: Conglomerado</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSCG0</entry>
<entry colname="col2">Could not create a template.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSCHx: Heap</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSCH0</entry>
<entry colname="col2">Could not create container.</entry>
</row>
<row>
<entry colname="col1">XSCH1 </entry>
<entry colname="col2">Container &lt;<i>containerName</i>> not found.  </entry>
</row>
<row>
<entry colname="col1">XSCH4 </entry>
<entry colname="col2">Conglomerate could not be created.</entry>
</row>
<row>
<entry colname="col1">XSCH5 </entry>
<entry colname="col2">In a base table there was a mismatch between the requested
column number &lt;<i>columnnumber</i>> and the maximum number of columns &lt;<i>maxcol</i>>.</entry>
</row>
<row>
<entry colname="col1">XSCH6 </entry>
<entry colname="col2">The heap container with container id &lt;<i>containerID</i>>
is closed. </entry>
</row>
<row>
<entry colname="col1">XSCH7 </entry>
<entry colname="col2">The scan is not positioned.</entry>
</row>
<row>
<entry colname="col1">XSCH8 </entry>
<entry colname="col2">The feature is not implemented.  </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSDAx: RawStore - Data.Generic statement
exceptions</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSDA1 </entry>
<entry colname="col2">An attempt was made to access an out of range slot on
a page.</entry>
</row>
<row>
<entry colname="col1">XSDA2 </entry>
<entry colname="col2">An attempt was made to update a deleted record.</entry>
</row>
<row>
<entry colname="col1">XSDA3 </entry>
<entry colname="col2">Limitation: Record cannot be updated or inserted due
to lack of space on the page. Use the parameters derby.storage.pageSize and/or
derby.storage.pageReservedSpace to work around this limitation.    </entry>
</row>
<row>
<entry colname="col1">XSDA4 </entry>
<entry colname="col2">An unexpected exception was thrown.</entry>
</row>
<row>
<entry colname="col1">XSDA5 </entry>
<entry colname="col2">An attempt was made to undelete a record that was not
deleted.</entry>
</row>
<row>
<entry colname="col1">XSDA6 </entry>
<entry colname="col2">Column &lt;<i>column</i>> of row is null, it needs to
be set to point to an object. </entry>
</row>
<row>
<entry colname="col1">XSDA7 </entry>
<entry colname="col2">Restore of a serializable or SQLData object of class &lt;<i>className</i>>,
attempted to read more data than was originally stored.   </entry>
</row>
<row>
<entry colname="col1">XSDA8 </entry>
<entry colname="col2">Exception during restore of a serializable or SQLData
object of class &lt;<i>className</i>>.</entry>
</row>
<row>
<entry colname="col1">XSDA9 </entry>
<entry colname="col2">Class not found during restore of a serializable or
SQLData object of class &lt;<i>className</i>>.</entry>
</row>
<row>
<entry colname="col1">XSDAA </entry>
<entry colname="col2">Illegal time stamp &lt;<i>timestamp</i>>, either time
stamp is from a different page or of incompatible implementation.</entry>
</row>
<row>
<entry colname="col1">XSDAB </entry>
<entry colname="col2">Cannot set a null time stamp.</entry>
</row>
<row>
<entry colname="col1">XSDAC </entry>
<entry colname="col2">Attempt to move either rows or pages from one container
to another. </entry>
</row>
<row>
<entry colname="col1">XSDAD </entry>
<entry colname="col2">Attempt to move zero rows from one page to another.
 </entry>
</row>
<row>
<entry colname="col1">XSDAE </entry>
<entry colname="col2">Can only make a record handle for special record handle
IDs.</entry>
</row>
<row>
<entry colname="col1">XSDAF </entry>
<entry colname="col2">Using special record handle as if it were a normal record
handle.</entry>
</row>
<row>
<entry colname="col1">XSDAG </entry>
<entry colname="col2">The allocation nested top transaction cannot open the
container</entry>
</row>
<row>
<entry colname="col1">XSDAI </entry>
<entry colname="col2">Page &lt;<i>page</i>> being removed is already locked
for deallocation.</entry>
</row>
<row>
<entry colname="col1">XSDAJ </entry>
<entry colname="col2">Exception during write of a serializable or SQLData
object.</entry>
</row>
<row>
<entry colname="col1">XSDAK </entry>
<entry colname="col2">The wrong page was retrieved for record handle &lt;<i>value</i>>. </entry>
</row>
<row>
<entry colname="col1">XSDAL </entry>
<entry colname="col2">Record handle &lt;<i>value</i>> unexpectedly points
to overflow page.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSDBx: RawStore - Data.Generic transaction
exceptions</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSDB0 </entry>
<entry colname="col2">Unexpected exception on in-memory page &lt;<i>page</i>>.</entry>
</row>
<row>
<entry colname="col1">XSDB1 </entry>
<entry colname="col2">Unknown page format at page &lt;<i>page</i>>.</entry>
</row>
<row>
<entry colname="col1">XSDB2 </entry>
<entry colname="col2">Unknown container format at container &lt;<i>containerName</i>>
: &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSDB3 </entry>
<entry colname="col2">Container information cannot change once written: was &lt;<i>value</i>>,
now &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSDB4 </entry>
<entry colname="col2">Page &lt;<i>page</i>> is at version &lt;<i>value</i>>
but the log file contains change version &lt;<i>value</i>>. Either the log
records for this page are missing or this page was not written to disk properly. </entry>
</row>
<row>
<entry colname="col1">XSDB5 </entry>
<entry colname="col2">Log has change record on page &lt;<i>page</i>>, which
is beyond the end of the container. </entry>
</row>
<row>
<entry colname="col1">XSDB6 </entry>
<entry colname="col2">Another instance of Derby may have already booted the
database &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSDB7 </entry>
<entry colname="col2">WARNING: Derby (instance &lt;<i>value</i>>) is attempting
to boot the database &lt;<i>value</i>> even though Derby (instance &lt;<i>value</i>>)
might still be active. Only one instance of Derby should boot a database at
a time. Severe and non-recoverable corruption can result and might have already
occurred. </entry>
</row>
<row>
<entry colname="col1">XSDB8</entry>
<entry colname="col2">WARNING: Derby (instance &lt;<i>value</i>>) is attempting
to boot the database &lt;<i>value</i>> even though Derby (instance &lt;<i>value</i>>)
might still be active. Only one instance of Derby should boot a database at
a time. Severe and non-recoverable corruption can result if two instances
of Derby boot on the same database at the same time. The db2j.database.forceDatabaseLock=true
property is set so the database will not boot until the db.lck is no longer
present. Normally this file is removed when the first instance of Derby to
boot on the database exits. However, it is not removed in some shutdowns.
If the file is not removed, you must remove it manually. It is important to
verify that no other VM is accessing the database before manually deleting
the db.lck file.   </entry>
</row>
<row>
<entry colname="col1">XSDB9 </entry>
<entry colname="col2">Stream container &lt;<i>containerName</i>> is corrupt.</entry>
</row>
<row>
<entry colname="col1">XSDBA </entry>
<entry colname="col2">Attempt to allocate object &lt;<i>object</i>> failed. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSDFx: RawStore - Data.Filesystem statement
exceptions</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSDF0 </entry>
<entry colname="col2">Could not create file &lt;<i>fileName</i>> as it already
exists. </entry>
</row>
<row>
<entry colname="col1">XSDF1 </entry>
<entry colname="col2">Exception during creation of file &lt;<i>fileName</i>>
for container.</entry>
</row>
<row>
<entry colname="col1">XSDF2 </entry>
<entry colname="col2">Exception during creation of file &lt;<i>fileName</i>>
for container, file could not be removed. The exception was: &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSDF3 </entry>
<entry colname="col2">Cannot create segment &lt;<i>segmentName</i>>. </entry>
</row>
<row>
<entry colname="col1">XSDF4 </entry>
<entry colname="col2">Exception during remove of file &lt;<i>fileName</i>>
for dropped container, file could not be removed &lt;<i>value</i>>.  </entry>
</row>
<row>
<entry colname="col1">XSDF6 </entry>
<entry colname="col2">Cannot find the allocation page &lt;<i>page</i>>.   </entry>
</row>
<row>
<entry colname="col1">XSDF7 </entry>
<entry colname="col2">Newly created page failed to be latched &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSDF8 </entry>
<entry colname="col2">Cannot find page &lt;<i>page</i>> to reuse. </entry>
</row>
<row>
<entry colname="col1">XSDFB </entry>
<entry colname="col2">Operation not supported by a read only database.</entry>
</row>
<row>
<entry colname="col1">XSDFD </entry>
<entry colname="col2">Different page image read on two I/Os on Page &lt;<i>page</i>>.
The first image has an incorrect checksum, the second image has a correct
checksum. Page images follows: &lt;<i>value</i>> &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSDFF </entry>
<entry colname="col2">The requested operation failed due to an unexpected
exception.  </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSDGx: RawStore - Data.Filesystem database
exceptions</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSDG0 </entry>
<entry colname="col2">Page &lt;<i>page</i>> could not be read from disk.
 </entry>
</row>
<row>
<entry colname="col1">XSDG1 </entry>
<entry colname="col2">Page &lt;<i>page</i>> could not be written to disk,
please check if disk is full.</entry>
</row>
<row>
<entry colname="col1">XSDG2 </entry>
<entry colname="col2">Invalid checksum on Page &lt;<i>page</i>>, expected=&lt;<i>value</i>>,
on-disk version=&lt;<i>value</i>>, page dump follows: &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSDG3 </entry>
<entry colname="col2">Meta-data for Container &lt;<i>containerName</i>> could
not be accessed.</entry>
</row>
<row>
<entry colname="col1">XSDG5 </entry>
<entry colname="col2">Database is not in create mode when createFinished is
called. </entry>
</row>
<row>
<entry colname="col1">XSDG6 </entry>
<entry colname="col2">Data segment directory not found in &lt;<i>value</i>>
backup during restore. Please make sure that backup copy is the right one
and it is not corrupted. </entry>
</row>
<row>
<entry colname="col1">XSDG7 </entry>
<entry colname="col2">Directory &lt;<i>directoryName</i>> could not be removed
during restore. Please make sure that permissions are correct. </entry>
</row>
<row>
<entry colname="col1">XSDG8 </entry>
<entry colname="col2">Unable to copy directory '&lt;<i>directoryName</i>>'
to '&lt;<i>value</i>>' during restore. Please make sure that there is enough
space and permissions are correct. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSLAx: RawStore - Log.Generic database exceptions</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSLA0 </entry>
<entry colname="col2">Cannot flush the log file to disk &lt;<i>value</i>>.
   </entry>
</row>
<row>
<entry colname="col1">XSLA1 </entry>
<entry colname="col2">Log Record has been sent to the stream, but it cannot
be applied to the store (Object &lt;<i>object</i>>).  This may cause recovery
problems also. </entry>
</row>
<row>
<entry colname="col1">XSLA2 </entry>
<entry colname="col2">An IOException occurred while accessing the log file.
The system will shut down.</entry>
</row>
<row>
<entry colname="col1">XSLA3 </entry>
<entry colname="col2">The log file is corrupt. The log stream contains invalid
data. </entry>
</row>
<row>
<entry colname="col1">XSLA4 </entry>
<entry colname="col2">Unable to write to the log, most likely because the
log is full. It is also possible that the file system is read-only, the disk
failed, or another problem occurred with the media. Delete unnecessary files.</entry>
</row>
<row>
<entry colname="col1">XSLA5 </entry>
<entry colname="col2">Cannot read log stream for some reason to rollback transaction &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSLA6 </entry>
<entry colname="col2">Cannot recover the database.</entry>
</row>
<row>
<entry colname="col1">XSLA7 </entry>
<entry colname="col2">Cannot redo operation &lt;<i>operation</i>> in the log.
  </entry>
</row>
<row>
<entry colname="col1">XSLA8 </entry>
<entry colname="col2">Cannot rollback transaction &lt;<i>value</i>>, trying
to compensate &lt;<i>value</i>> operation with &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSLAA </entry>
<entry colname="col2">The store has been marked for shutdown by an earlier
exception. </entry>
</row>
<row>
<entry colname="col1">XSLAB </entry>
<entry colname="col2">Cannot find log file &lt;<i>logfileName</i>>. Verify
that the logDevice property is set with the correct path separator for your
platform.</entry>
</row>
<row>
<entry colname="col1">XSLAC </entry>
<entry colname="col2">Database at &lt;<i>value</i>> have incompatible format
with the current version of software, it may have been created by or upgraded
by a later version. </entry>
</row>
<row>
<entry colname="col1">XSLAD </entry>
<entry colname="col2">Log record at instance &lt;<i>value</i>> in log file &lt;<i>logfileName</i>>
is corrupted. Expected log record length &lt;<i>value</i>>, actual length &lt;<i>value</i>>. </entry>
</row>
<row>
<entry colname="col1">XSLAE </entry>
<entry colname="col2">Control file at &lt;<i>value</i>> cannot be written
or updated.</entry>
</row>
<row>
<entry colname="col1">XSLAF </entry>
<entry colname="col2">A read-only database was created with dirty data buffers.</entry>
</row>
<row>
<entry colname="col1">XSLAH</entry>
<entry colname="col2">A read-only database is being updated. </entry>
</row>
<row>
<entry colname="col1">XSLAI </entry>
<entry colname="col2">Cannot log the checkpoint log record.</entry>
</row>
<row>
<entry colname="col1">XSLAJ </entry>
<entry colname="col2">The log record size &lt;<i>value</i>> exceeds the maximum
allowable log file size &lt;<i>maxSize</i>>. An error was encountered in log
file &lt;<i>fileName</i>>, position &lt;<i>value</i>>. </entry>
</row>
<row>
<entry colname="col1">XSLAK </entry>
<entry colname="col2">Database has exceeded largest log file number &lt;<i>value</i>>.
 </entry>
</row>
<row>
<entry colname="col1">XSLAL </entry>
<entry colname="col2">The log record size &lt;<i>value</i>> exceeds the maximum
allowable log file size &lt;<i>maxSize</i>>. An error was encountered in log
file &lt;<i>fileName</i>>, position &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSLAM </entry>
<entry colname="col2">Cannot verify database format at &lt;<i>value</i>> due
to IOException. </entry>
</row>
<row>
<entry colname="col1">XSLAN </entry>
<entry colname="col2">Database at &lt;<i>value</i>> has an incompatible format
with the current version of the software. The database was created by or upgraded
by version &lt;<i>version</i>>. </entry>
</row>
<row>
<entry colname="col1">XSLAO </entry>
<entry colname="col2">Recovery failed. Unexpected problem &lt;<i>value</i>>. </entry>
</row>
<row>
<entry colname="col1">XSLAP </entry>
<entry colname="col2">Database at &lt;<i>value</i>> is at version &lt;<i>version</i>>.
Beta databases cannot be upgraded.</entry>
</row>
<row>
<entry colname="col1">XSLAQ </entry>
<entry colname="col2">Cannot create log file at directory &lt;<i>directory</i>>. </entry>
</row>
<row>
<entry colname="col1">XSLAR </entry>
<entry colname="col2">Unable to copy log file '&lt;<i>logfileName</i>>' to
'&lt;<i>value</i>>' during restore. Please make sure that there is enough
space and permissions are correct.</entry>
</row>
<row>
<entry colname="col1">XSLAS </entry>
<entry colname="col2">Log directory &lt;<i>directory</i>> not found in backup
during restore. Please make sure that backup copy is the correct one and it
is not corrupted.</entry>
</row>
<row>
<entry colname="col1">XSLAT </entry>
<entry colname="col2">Log directory &lt;<i>directory</i>> exists. Please make
sure specified logDevice location is correct. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSLBx: RawStore - Log.Generic statement exceptions</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSLB1 </entry>
<entry colname="col2">Log operation &lt;<i>logoperation</i>> encounters error
writing itself out to the log stream, this could be caused by an errant log
operation or internal log buffer full due to excessively large log operation. </entry>
</row>
<row>
<entry colname="col1">XSLB2 </entry>
<entry colname="col2">Log operation &lt;<i>logoperation</i>> logging excessive
data, it filled up the internal log buffer.</entry>
</row>
<row>
<entry colname="col1">XSLB4 </entry>
<entry colname="col2">Cannot find truncationLWM &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSLB5 </entry>
<entry colname="col2">Illegal truncationLWM instance &lt;<i>value</i>> for
truncation point &lt;<i>value</i>>. Legal range is from &lt;<i>value</i>>
to &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSLB6 </entry>
<entry colname="col2">Trying to log a 0 or -ve length log Record.</entry>
</row>
<row>
<entry colname="col1">XSLB8 </entry>
<entry colname="col2">Trying to reset a scan to &lt;<i>value</i>>, beyond
its limit of &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSLB9 </entry>
<entry colname="col2">Unable to issue any more changes. Log factory has been
stopped. </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSRSx: RawStore - protocol.Interface statement
exceptions</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSRS0 </entry>
<entry colname="col2">Cannot freeze the database after it is already frozen.
 </entry>
</row>
<row>
<entry colname="col1">XSRS1 </entry>
<entry colname="col2">Cannot backup the database to &lt;<i>value</i>>, which
is not a directory. </entry>
</row>
<row>
<entry colname="col1">XSRS4</entry>
<entry colname="col2">Error renaming file (during backup) from &lt;<i>value</i>>
to &lt;<i>value</i>>.</entry>
</row>
<row>
<entry colname="col1">XSRS5 </entry>
<entry colname="col2">Error copying file (during backup) from &lt;<i>path</i>>
to &lt;<i>path</i>>.   </entry>
</row>
<row>
<entry colname="col1">XSRS6 </entry>
<entry colname="col2">Cannot create backup directory &lt;<i>directoryName</i>>.
   </entry>
</row>
<row>
<entry colname="col1">XSRS7 </entry>
<entry colname="col2">Backup caught unexpected exception. </entry>
</row>
<row>
<entry colname="col1">XSRS8 </entry>
<entry colname="col2">Log device can only be set during database creation
time, it cannot be changed after the database is created.</entry>
</row>
<row>
<entry colname="col1">XSRS9 </entry>
<entry colname="col2">Record &lt;<i>recordName</i>> no longer exists.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSTA2: XACT_TRANSACTION_ACTIVE</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSTA2 </entry>
<entry colname="col2">A transaction was already active when an attempt was
made to activate another transaction.</entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XSTBx: RawStore - Transactions.Basic system
exceptions</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XSTB0 </entry>
<entry colname="col2">An exception was thrown during transaction abort. </entry>
</row>
<row>
<entry colname="col1">XSTB2 </entry>
<entry colname="col2">Unable to log transaction changes, possibly because
the database is read-only.</entry>
</row>
<row>
<entry colname="col1">XSTB3 </entry>
<entry colname="col2">Cannot abort transaction because the log manager is
null, probably due to an earlier error.   </entry>
</row>
<row>
<entry colname="col1">XSTB5 </entry>
<entry colname="col2">Creating database with logging disabled encountered
unexpected problem.</entry>
</row>
<row>
<entry colname="col1">XSTB6 </entry>
<entry colname="col2">Cannot substitute a transaction table with another while
one is already in use.  </entry>
</row>
</tbody>
</tgroup>
</table>
<table>
<title>Código de classe XXXXX : No SQLSTATE</title>
<tgroup cols="2">
<colspec colname="col1" colwidth="128*"/>
<colspec colname="col2"
colwidth="674*"/>
<thead>
<row>
<entry colname="col1">SQLSTATE </entry>
<entry colname="col2">Texto da mensagem</entry>
</row>
</thead>
<tbody>
<row>
<entry colname="col1">XXXXX </entry>
<entry colname="col2">Normal database session close. </entry>
</row>
</tbody>
</tgroup>
</table>
</section>
</refbody>
</reference>
